<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0053)https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hhmipa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2898524/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Face Recognition by Independent Component Analysis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/jig.min.css"><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/jig.min.js"></script><style></style>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="IEEE transactions on neural networks / a publication of the IEEE Neural Networks Council"><meta name="citation_title" content="Face Recognition by Independent Component Analysis"><meta name="citation_authors" content="Marian Stewart Bartlett, Javier R. Movellan, Terrence J. Sejnowski"><meta name="citation_date" content="2002"><meta name="citation_issue" content="6"><meta name="citation_volume" content="13"><meta name="citation_firstpage" content="1450"><meta name="citation_doi" content="10.1109/TNN.2002.804287"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2898524/?report=abstract"><meta name="citation_pmid" content="18244540"><meta name="DC.Title" content="Face Recognition by Independent Component Analysis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Howard Hughes Medical Institute"><meta name="DC.Contributor" content="Marian Stewart Bartlett"><meta name="DC.Contributor" content="Javier R. Movellan"><meta name="DC.Contributor" content="Terrence J. Sejnowski"><meta name="DC.Date" content="2002"><meta name="DC.Identifier" content="10.1109/TNN.2002.804287"><meta name="DC.Language" content="en"><meta property="og:title" content="Face Recognition by Independent Component Analysis"><meta property="og:type" content="article"><meta property="og:description" content="A number of current face recognition algorithms use face representations found by unsupervised statistical methods. Typically these methods find a set of basis images and represent faces as a linear combination of those images. Principal component analysis ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/common.min.js">//</script><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:&#39;/pmc/utils/ctxp/&#39;"><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/jquery.citationexporter.js">//</script><link rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/MathJax.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhmipa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/epub/"><link rel="alternate" type="application/pdf" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/pdf/nihms-199383.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="https://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="F4FC6255ACB279F100000000007B0079.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/4143404.css"><link type="text/css" rel="stylesheet" href="./Face Recognition by Independent Component Analysis_files/4157116.css" media="print"><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('https://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC2898524%2F&ncbi_pdid=article&ncbi_phid=F4FC6255ACB279F100000000007B0079.m_8'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	&lt;p class="nojs"&gt;
	&lt;strong&gt;Warning:&lt;/strong&gt;
	The NCBI web site requires JavaScript to function. 
	&lt;a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank"&gt;more...&lt;/a&gt;
	&lt;/p&gt;
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="https://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./Face Recognition by Independent Component Analysis_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="https://www.ncbi.nlm.nih.gov/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="https://www.ncbi.nlm.nih.gov/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F" id="sign_in">Sign in to NCBI</a><a href="https://www.ncbi.nlm.nih.gov/account/signout/?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="https://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="https://www.ncbi.nlm.nih.gov/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:&#39;pmc-search-autocomplete&#39;,disableUrl:&#39;NcbiSearchBarAutoComplCtrl&#39;" autocomplete="off" data-sbconfig="ds:&#39;no&#39;,pjs:&#39;no&#39;,afs:&#39;yes&#39;" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" style="visibility: hidden;"><img src="./Face Recognition by Independent Component Analysis_files/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=hhmi%20author%20manuscript[filter]" class="navlink">HHMI Author Manuscripts</a></li><li class="accid">PMC2898524</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./Face Recognition by Independent Component Analysis_files/logo-hhmipa.jpg" alt="Logo of hhmipa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,58,250,75" alt="about author manuscripts" title="about author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="251,58,500,75" alt="submit a manuscript" title="submit a manuscript" href="http://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHMI Howard Hughes Medical Institute; Author Manuscript; Accepted for publication in peer reviewed journal" title="HHMI Howard Hughes Medical Institute; Author Manuscript; Accepted for publication in peer reviewed journal" href="http://www.hhmi.org/about/research/papp.html" ref="reftype=publisher&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" role="menuitem" aria-expanded="false" aria-haspopup="true">IEEE Trans Neural Netw</a></span>. Author manuscript; available in PMC 2010 Jul 7.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18244540" target="pmc_ext" ref="reftype=publisher&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal"><span class="cit">IEEE Trans Neural Netw. 2002; 13(6): 1450–1464. </span></a></div> <span class="doi">doi:&nbsp; <a href="https://dx.doi.org/10.1109%2FTNN.2002.804287" target="pmc_ext" ref="reftype=other&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.1109/TNN.2002.804287</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2898524</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">HHMIMSID: </span><span>HHMIMS199383</span></div><div class="fm-citation-pmid">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18244540">18244540</a></div></div></div></div></div><h1 class="content-title">Face Recognition by Independent Component Analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Bartlett%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18244540">Marian Stewart Bartlett</a>, <span class="fm-role">Member, IEEE</span>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Movellan%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18244540">Javier R. Movellan</a>, <span class="fm-role">Member, IEEE</span>, and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sejnowski%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18244540">Terrence J. Sejnowski</a>, <span class="fm-role">Fellow, IEEE</span></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140094232724976_ai" style="display:none"><div class="fm-affl" lang="en" id="A1">University of California-San Diego, La Jolla, CA 92093-0523 USA (<a href="mailto:dev@null" data-email="ude.klas@inram" class="oemail">ude.klas@inram</a>; <a href="mailto:dev@null" data-email="ude.dscu.cni@reivaj" class="oemail">ude.dscu.cni@reivaj</a>; <a href="mailto:dev@null" data-email="ude.klas@yrret" class="oemail">ude.klas@yrret</a>).</div><div id="FN1">T. J. Sejnowski is also with the Howard Hughes Medical Institute at the Salk Institute, La Jolla, CA 92037 USA.</div></div><div class="togglers"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="pmctoggle" rid="idm140094232724976_ai">Author information <span>►</span></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="pmctoggle" rid="idm140094232724976_cpl">Copyright and License information <span>►</span></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="idm140094232724976_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18244540" target="pmc_ext" ref="reftype=publisher&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=normal">IEEE Trans Neural Netw</a></div><div>See other articles in PMC that <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140094232838208" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140094232838208title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">A number of current face recognition algorithms use face representations found by unsupervised statistical methods. Typically these methods find a set of basis images and represent faces as a linear combination of those images. Principal component analysis (PCA) is a popular example of such methods. The basis images found by PCA depend only on pairwise relationships between pixels in the image database. In a task such as face recognition, in which important information may be contained in the high-order relationships among pixels, it seems reasonable to expect that better basis images may be found by methods sensitive to these high-order statistics. Independent component analysis (ICA), a generalization of PCA, is one such method. We used a version of ICA derived from the principle of optimal information transfer through sigmoidal neurons. ICA was performed on face images in the FERET database under two different architectures, one which treated the images as random variables and the pixels as outcomes, and a second which treated the pixels as random variables and the images as outcomes. The first architecture found spatially local basis images for the faces. The second architecture produced a factorial face code. Both ICA representations were superior to representations based on PCA for recognizing faces across days and changes in expression. A classifier that combined the two ICA representations gave the best performance.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Eigenfaces, face recognition, independent component analysis (ICA), principal component analysis (PCA), unsupervised learning</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">I. Introduction</h2><p id="P3" class="p p-first">REDUDENCY in the sensory input contains structural information about the environment. Barlow has argued that such redundancy provides knowledge [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>] and that the role of the sensory system is to develop factorial representations in which these dependencies are separated into independent components (ICs). Barlow also argued that such representations are advantageous for encoding complex objects that are characterized by high-order dependencies. Atick and Redlich have also argued for such representations as a general coding strategy for the visual system [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R3" rid="R3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a>].</p><p id="P4">Principal component analysis (PCA) is a popular unsupervised statistical method to find useful image representations. Consider a set of <em>n</em> basis images each of which has <em>n</em> pixels. A standard basis set consists of a single active pixel with intensity 1, where each basis image has a different active pixel. Any given image with <em>n</em> pixels can be decomposed as a linear combination of the standard basis images. In fact, the pixel values of an image can then be seen as the coordinates of that image with respect to the standard basis. The goal in PCA is to find a “better” set of basis images so that in this new basis, the image coordinates (the PCA coefficients) are uncorrelated, i.e., they cannot be linearly predicted from each other. PCA can, thus, be seen as partially implementing Barlow's ideas: Dependencies that show up in the joint distribution of pixels are separated out into the marginal distributions of PCA coefficients. However, PCA can only separate pairwise linear dependencies between pixels. High-order dependencies will still show in the joint distribution of PCA coefficients, and, thus, will not be properly separated.</p><p id="P5">Some of the most successful representations for face recognition, such as eigenfaces [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_367027317" role="button" aria-expanded="false" aria-haspopup="true">57</a>], holons [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R15" rid="R15" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">15</a>], and local feature analysis [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R50" rid="R50" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">50</a>] are based on PCA. In a task such as face recognition, much of the important information may be contained in the high-order relationships among the image pixels, and thus, it is important to investigate whether generalizations of PCA which are sensitive to high-order relationships, not just second-order relationships, are advantageous. Independent component analysis (ICA) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R14" rid="R14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a>] is one such generalization. A number of algorithms for performing ICA have been proposed. See [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R20" rid="R20" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">20</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R29" rid="R29" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">29</a>] for reviews. Here, we employ an algorithm developed by Bell and Sejnowski [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510565" role="button" aria-expanded="false" aria-haspopup="true">11</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510577" role="button" aria-expanded="false" aria-haspopup="true">12</a>] from the point of view of optimal information transfer in neural networks with sigmoidal transfer functions. This algorithm has proven successful for separating randomly mixed auditory signals (the cocktail party problem), and for separating electroencephalogram (EEG) signals [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R37" rid="R37" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">37</a>] and functional magnetic resonance imaging (fMRI) signals [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510564" role="button" aria-expanded="false" aria-haspopup="true">39</a>].</p><p id="P6">We performed ICA on the image set under two architectures. Architecture I treated the images as random variables and the pixels as outcomes, whereas Architecture II treated the pixels as random variables and the images as outcomes.<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN2" rid="FN2" class=" fn">1</a></sup> Matlab code for the ICA representations is available at <a href="http://inc.ucsd.edu/~marni" ref="reftype=extlink&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal" target="pmc_ext">http://inc.ucsd.edu/~marni</a>.</p><p id="P7" class="p p-last">Face recognition performance was tested using the FERET database [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R52" rid="R52" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">52</a>]. Face recognition performances using the ICA representations were benchmarked by comparing them to performances using PCA, which is equivalent to the “eigenfaces” representation [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R51" rid="R51" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">51</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_367027318" role="button" aria-expanded="false" aria-haspopup="true">57</a>]. The two ICA representations were then combined in a single classifier.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">II. ICA</h2><p id="P8" class="p p-first">There are a number of algorithms for performing ICA [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510560" role="button" aria-expanded="false" aria-haspopup="true">11</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R13" rid="R13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R14" rid="R14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R25" rid="R25" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">25</a>]. We chose the infomax algorithm proposed by Bell and Sejnowski [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510588" role="button" aria-expanded="false" aria-haspopup="true">11</a>], which was derived from the principle of optimal information transfer in neurons with sigmoidal transfer functions [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937472" role="button" aria-expanded="false" aria-haspopup="true">27</a>]. The algorithm is motivated as follows: Let <strong>X</strong> be an <em>n</em>-dimensional (<em>n</em>-D) random vector representing a distribution of inputs in the environment. (Here, boldface capitals denote random variables, whereas plain text capitals denote matrices). Let <em>W</em> be an <em>n</em> × <em>n</em> invertible matrix, <strong>U</strong> = <em>W</em><strong>X</strong> and <strong>Y</strong> = <em>f</em>(<strong>U</strong>) an <em>n</em>-D random variable representing the outputs of <em>n</em>-neurons. Each component of <em>f</em> = (<em>f</em><sub>1</sub>, …, <em>f<sub>n</sub></em>) is an invertible squashing function, mapping real numbers into the [0, 1] interval. Typically, the logistic function is used
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 8.664em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.794em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.987em, 1006.75em, 3.349em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="msub" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.45em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-5" style="font-family: STIXGeneral; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.15em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.298em;"><span class="mi" id="MathJax-Span-6" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-7" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-8" style="font-family: STIXGeneral;">(</span><span class="mi" id="MathJax-Span-9" style="font-family: STIXGeneral; font-style: italic;">u</span><span class="mo" id="MathJax-Span-10" style="font-family: STIXGeneral;">)</span></span><span class="mo" id="MathJax-Span-11" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mfrac" id="MathJax-Span-12" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 3.152em; height: 0px; margin-right: 0.101em; margin-left: 0.101em;"><span style="position: absolute; clip: rect(3.152em, 1000.4em, 4.136em, -999.998em); top: -4.673em; left: 50%; margin-left: -0.244em;"><span class="mn" id="MathJax-Span-13" style="font-family: STIXGeneral;">1</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.152em, 1003.05em, 4.186em, -999.998em); top: -3.295em; left: 50%; margin-left: -1.523em;"><span class="mrow" id="MathJax-Span-14"><span class="mn" id="MathJax-Span-15" style="font-family: STIXGeneral;">1</span><span class="mo" id="MathJax-Span-16" style="font-family: STIXGeneral; padding-left: 0.249em;">+</span><span class="msup" id="MathJax-Span-17" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 1.38em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.4em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-18" style="font-family: STIXGeneral; font-style: italic;">e</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.279em; left: 0.445em;"><span class="mrow" id="MathJax-Span-19"><span class="mo" id="MathJax-Span-20" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mi" id="MathJax-Span-21" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">u</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(0.888em, 1003.15em, 1.233em, -999.998em); top: -1.277em; left: 0.002em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.002em; border-top: 1.3px solid; width: 3.152em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-22" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.997em; border-left: 0px solid; width: 0px; height: 2.753em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><msub><mi>f</mi><mi>i</mi></msub><mrow><mo>(</mo><mi>u</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><msup><mi>e</mi><mrow><mo>−</mo><mi>u</mi></mrow></msup></mrow></mfrac><mo>.</mo></mrow></math></script></div><div class="l">(1)</div></div><p>
The <strong>U</strong><sub>1</sub>, …, <strong>U</strong><sub><em>n</em></sub> variables are linear combinations of inputs and can be interpreted as presynaptic activations of <em>n</em>-neurons. The <strong>Y</strong><sub>1</sub>, …, <strong>Y</strong><sub><em>n</em></sub> variables can be interpreted as postsynaptic activation rates and are bounded by the interval [0, 1]. The goal in Bell and Sejnowski's algorithm is to maximize the mutual information between the environment <strong>X</strong> and the output of the neural network <strong>Y</strong>. This is achieved by performing gradient ascent on the entropy of the output with respect to the weight matrix <em>W</em>. The gradient update rule for the weight matrix, <em>W</em> is as follows:
</p><div class="disp-formula" id="FD2"><div class="f">Δ<em>W</em>&nbsp;∝&nbsp;∇<sub><em>W</em></sub><em>H</em>(<strong>Y</strong>) =&nbsp;(<em>W</em><sup><em>T</em></sup>)<sup>−1</sup> +&nbsp;<em>E</em>(<strong>Y</strong>′<strong>X</strong><sup><em>T</em></sup>)</div><div class="l">(2)</div></div><p>
where <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M3" overflow="scroll" role="math" style="width: 11.321em; display: inline-block;"><span style="display: inline-block; position: relative; width: 8.91em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.873em, 1008.86em, 3.398em, -999.998em); top: -2.901em; left: 0.002em;"><span class="mrow" id="MathJax-Span-24"><span class="mrow" id="MathJax-Span-25"><span class="msubsup" id="MathJax-Span-26"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.69em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-27" style="font-family: STIXGeneral; font-weight: bold;">Y</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.447em, 1000.3em, 4.136em, -999.998em); top: -4.328em; left: 0.741em;"><span class="mo" id="MathJax-Span-28" style="font-size: 70.7%; font-family: STIXVariants;">′</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.3em, 4.136em, -999.998em); top: -3.688em; left: 0.741em;"><span class="mi" id="MathJax-Span-29" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-30" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msubsup" id="MathJax-Span-31" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.45em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-32" style="font-family: STIXGeneral; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.15em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.447em, 1000.5em, 4.136em, -999.998em); top: -4.328em; left: 0.544em;"><span class="mo" id="MathJax-Span-33" style="font-size: 70.7%; font-family: STIXVariants;">″</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.3em, 4.136em, -999.998em); top: -3.688em; left: 0.298em;"><span class="mi" id="MathJax-Span-34" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-35" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-36" style="font-family: STIXGeneral;">(</span><span class="msub" id="MathJax-Span-37"><span style="display: inline-block; position: relative; width: 0.987em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.69em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-38" style="font-family: STIXGeneral; font-weight: bold;">U</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.741em;"><span class="mi" id="MathJax-Span-39" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-40" style="font-family: STIXGeneral;">)</span></span><span class="mo" id="MathJax-Span-41" style="vertical-align: -0.194em; padding-left: 0.249em;"><span style="font-family: STIXSizeOneSym;">/</span></span><span class="msubsup" id="MathJax-Span-42" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 0.839em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.45em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-43" style="font-family: STIXGeneral; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.15em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.447em, 1000.3em, 4.136em, -999.998em); top: -4.328em; left: 0.544em;"><span class="mo" id="MathJax-Span-44" style="font-size: 70.7%; font-family: STIXVariants;">′</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.3em, 4.136em, -999.998em); top: -3.688em; left: 0.298em;"><span class="mi" id="MathJax-Span-45" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-46" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-47" style="font-family: STIXGeneral;">(</span><span class="msub" id="MathJax-Span-48"><span style="display: inline-block; position: relative; width: 0.987em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.69em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-49" style="font-family: STIXGeneral; font-weight: bold;">U</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.741em;"><span class="mi" id="MathJax-Span-50" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-51" style="font-family: STIXGeneral;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.906em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.497em; border-left: 0px solid; width: 0px; height: 1.691em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M3" overflow="scroll"><mrow><msubsup><mi mathvariant="bold">Y</mi><mi>i</mi><mo>′</mo></msubsup><mo>=</mo><msubsup><mi>f</mi><mi>i</mi><mo>″</mo></msubsup><mrow><mo>(</mo><msub><mi mathvariant="bold">U</mi><mi>i</mi></msub><mo>)</mo></mrow><mo>∕</mo><msubsup><mi>f</mi><mi>i</mi><mo>′</mo></msubsup><mrow><mo>(</mo><msub><mi mathvariant="bold">U</mi><mi>i</mi></msub><mo>)</mo></mrow></mrow></math></script>, the ratio between the second and first partial derivatives of the activation function, <em>T</em> stands for transpose, <em>E</em> for expected value, <em>H</em>(<strong>Y</strong>) is the entropy of the random vector <strong>Y</strong>, and ▽<em><sub>W</sub>H</em>(<strong>Y</strong>) is the gradient of the entropy in matrix form, i.e., the cell in row <em>i</em>, column <em>j</em> of this matrix is the derivative of <em>H</em>(<strong>Y</strong>) with respect to <em>W<sub>ij</sub></em>. Computation of the matrix inverse can be avoided by employing the natural gradient [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R1" rid="R1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>], which amounts to multiplying the absolute gradient by <em>W<sup>T</sup>W</em>, resulting in the following learning rule [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510572" role="button" aria-expanded="false" aria-haspopup="true">12</a>]:
</p><div class="disp-formula" id="FD3"><div class="f">Δ<em>W</em>&nbsp;∝&nbsp;∇<sub><em>W</em></sub><em>H</em>(<strong>Y</strong>)<em>W</em><sup><em>T</em></sup><em>W</em> =&nbsp;(<em>I</em> +&nbsp;<strong>Y</strong>′<strong>U</strong><sup><em>T</em></sup>)<em>W</em></div><div class="l">(3)</div></div><p>
where <em>I</em> is the identity matrix. The logistic transfer function (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD1" rid="FD1" class=" disp-formula">1</a>) gives <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M5" overflow="scroll" role="math" style="width: 7.778em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.105em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.577em, 1006.06em, 2.906em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-53"><span class="mrow" id="MathJax-Span-54"><span class="msubsup" id="MathJax-Span-55"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.69em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-56" style="font-family: STIXGeneral; font-weight: bold;">Y</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.447em, 1000.3em, 4.136em, -999.998em); top: -4.328em; left: 0.741em;"><span class="mo" id="MathJax-Span-57" style="font-size: 70.7%; font-family: STIXVariants;">′</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.3em, 4.136em, -999.998em); top: -3.688em; left: 0.741em;"><span class="mi" id="MathJax-Span-58" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-59" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mrow" id="MathJax-Span-60" style="padding-left: 0.298em;"><span class="mo" id="MathJax-Span-61" style="font-family: STIXGeneral;">(</span><span class="mn" id="MathJax-Span-62" style="font-family: STIXGeneral;">1</span><span class="mo" id="MathJax-Span-63" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mn" id="MathJax-Span-64" style="font-family: STIXGeneral; padding-left: 0.249em;">2</span><span class="msub" id="MathJax-Span-65"><span style="display: inline-block; position: relative; width: 0.987em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.69em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-66" style="font-family: STIXGeneral; font-weight: bold;">Y</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.741em;"><span class="mi" id="MathJax-Span-67" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-68" style="font-family: STIXGeneral;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.441em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M5" overflow="scroll"><mrow><msubsup><mi mathvariant="bold">Y</mi><mi>i</mi><mo>′</mo></msubsup><mo>=</mo><mrow><mo>(</mo><mn>1</mn><mo>−</mo><mn>2</mn><msub><mi mathvariant="bold">Y</mi><mi>i</mi></msub><mo>)</mo></mrow></mrow></math></script>.</p><p id="P9">When there are multiple inputs and outputs, maximizing the joint entropy of the output <strong>Y</strong> encourages the individual outputs to move toward statistical independence. When the form of the nonlinear transfer function <em>f</em> is the same as the cumulative density functions of the underlying ICs (up to scaling and translation) it can be shown that maximizing the joint entropy of the outputs in <strong>Y</strong> also minimizes the mutual information between the individual outputs in <strong>U</strong> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510580" role="button" aria-expanded="false" aria-haspopup="true">12</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R42" rid="R42" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">42</a>]. In practice, the logistic transfer function has been found sufficient to separate mixtures of natural signals with sparse distributions including sound sources [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510561" role="button" aria-expanded="false" aria-haspopup="true">11</a>].</p><p id="P10" class="p">The algorithm is speeded up by including a “sphering” step prior to learning [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510559" role="button" aria-expanded="false" aria-haspopup="true">12</a>]. The row means of <strong>X</strong> are subtracted, and then <strong>X</strong> is passed through the whitening matrix <em>W<sub>z</sub></em>, which is twice the inverse square root<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN3" rid="FN3" class=" fn">2</a></sup> of the covariance matrix
</p><div class="disp-formula" id="FD4"><div class="f"><em>W</em><sub><em>z</em></sub> =&nbsp;2 ∗ (<em>Cov</em>(<strong>X</strong>))<sup>−(1∕2)</sup>.</div><div class="l">(4)</div></div><p>
This removes the first and the second-order statistics of the data; both the mean and covariances are set to zero and the variances are equalized. When the inputs to ICA are the “sphered” data, the full transform matrix <em>W<sub>I</sub></em> is the product of the sphering matrix and the matrix learned by ICA
</p><div class="disp-formula" id="FD5"><div class="f"><em>W</em><sub><em>I</em></sub> =&nbsp;<em>W</em><em>W</em><sub><em>z</em></sub>.</div><div class="l">(5)</div></div><p>
MacKay [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R36" rid="R36" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">36</a>] and Pearlmutter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R48" rid="R48" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">48</a>] showed that the ICA algorithm converges to the maximum likelihood estimate of <em>W</em><sup>−1</sup> for the following generative model of the data:
</p><div class="disp-formula" id="FD6"><div class="f"><strong>X</strong> =&nbsp;<em>W</em><sup>−1</sup><strong>S</strong></div><div class="l">(6)</div></div><p>
where <strong>S</strong> = (<strong>S</strong><sub>1</sub>, …, <strong>S</strong><sub><em>n</em></sub>)′ is a vector of independent random variables, called the sources, with cumulative distributions equal to <em>f<sub>i</sub></em>, in other words, using logistic activation functions corresponds to assuming logistic random sources and using the standard cumulative Gaussian distribution as activation functions corresponds to assuming Gaussian random sources. Thus, <em>W</em><sup>−1</sup>, the inverse of the weight matrix in Bell and Sejnowski's algorithm, can be interpreted as the source mixing matrix and the <strong>U</strong> = <em>W</em><strong>X</strong> variables can be interpreted as the maximum-likelihood (ML) estimates of the sources that generated the data.</p><div id="S3" class="sec"><h3>A. ICA and Other Statistical Techniques</h3><div id="S4" class="sec sec-first"><p></p><h4 class="inline">ICA and PCA </h4><p id="P11" class="p p-first">PCA can be derived as a special case of ICA which uses Gaussian source models. In such case the mixing matrix <em><strong>W</strong></em> is unidentifiable in the sense that there is an infinite number of equally good ML solutions. Among all possible ML solutions, PCA chooses an orthogonal matrix which is optimal in the following sense: 1) Regardless of the distribution of <strong>X</strong>, <strong>U</strong><sub>1</sub> is the linear combination of input that allows optimal linear reconstruction of the input in the mean square sense; and 2) for <strong>U</strong><sub>1</sub>, … <strong>U</strong><em><sub>k</sub></em> fixed, <strong>U</strong><sub><em>k</em>+1</sub> allows optimal linear reconstruction among the class of linear combinations of <strong>X</strong> which are uncorrelated with <strong>U</strong><sub>1</sub>, … <strong>U</strong><em><sub>k</sub></em>. If the sources are Gaussian, the likelihood of the data depends only on first- and second-order statistics (the covariance matrix). In PCA, the rows of <em>W</em> are, in fact, the eigenvectors of the covariance matrix of the data.</p><p id="P12">Second-order statistics capture the amplitude spectrum of images but not their phase spectrum. The high-order statistics capture the phase spectrum [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510582" role="button" aria-expanded="false" aria-haspopup="true">12</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>]. For a given sample of natural images, we can scramble their phase spectrum while maintaining their power spectrum. This will dramatically alter the appearance of the images but will not change their second-order statistics. The phase spectrum, not the power spectrum, contains the structural information in images that drives human perception. For example, as illustrated in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, a face image synthesized from the amplitude spectrum of face A and the phase spectrum of face B will be perceived as an image of face B [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R45" rid="R45" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">45</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510569" role="button" aria-expanded="false" aria-haspopup="true">53</a>]. The fact that PCA is only sensitive to the power spectrum of images suggests that it might not be particularly well suited for representing natural images.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094227031360" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0004.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0004.jpg"></div></a><div id="largeobj_idm140094227031360" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a4--><div><span>(left) Two face images. (Center) The two faces with scrambled phase. (right) Reconstructions with the amplitude of the original face and the phase of the other face. Faces images are from the FERET face database, reprinted with permission from J. Phillips.</span></div></div></div><p id="P13">The assumption of Gaussian sources implicit in PCA makes it inadequate when the true sources are non-Gaussian. In particular, it has been empirically observed that many natural signals, including speech, natural images, and EEG are better described as linear combinations of sources with long tailed distributions [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510563" role="button" aria-expanded="false" aria-haspopup="true">11</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>]. These sources are called “high-kurtosis,” “sparse,” or “super-Gaussian” sources. Logistic random variables are a special case of sparse source models. When sparse source models are appropriate, ICA has the following potential advantages over PCA: 1) It provides a better probabilistic model of the data, which better identifies where the data concentrate in <em>n</em>-dimensional space. 2) It uniquely identifies the mixing matrix <em>W</em>. 3) It finds a not-necessarily orthogonal basis which may reconstruct the data better than PCA in the presence of noise. 4) It is sensitive to high-order statistics in the data, not just the covariance matrix.</p><p id="P14"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> illustrates these points with an example. The figure shows samples from a three-dimensional (3-D) distribution constructed by linearly mixing two high-kurtosis sources. The figure shows the basis vectors found by PCA and by ICA on this problem. Since the three ICA basis vectors are nonorthogonal, they change the relative distance between data points. This change in metric may be potentially useful for classification algorithms, like nearest neighbor, that make decisions based on relative distances between points. The ICA basis also alters the angles between data points, which affects similarity measures such as cosines. Moreover, if an undercomplete basis set is chosen, PCA and ICA may span different subspaces. For example, in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, when only two dimensions are selected, PCA and ICA choose different subspaces.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094228507712" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0005.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0005.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0005.jpg"></a></div><div id="largeobj_idm140094228507712" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a4--><div><span>(top) Example 3-D data distribution and corresponding PC and IC axes. Each axis is a column of the mixing matrix <em>W</em><sup>−1</sup> found by PCA or ICA. Note the PC axes are orthogonal while the IC axes are not. If only two components are allowed, ICA chooses a different subspace than PCA. (bottom left) Distribution of the first PCA coordinates of the data. (bottom right) Distribution of the first ICA coordinates of the data. Note that since the ICA axes are nonorthogonal, relative distances between points are different in PCA than in ICA, as are the angles between points.</span></div></div></div><p id="P15">The metric induced by ICA is superior to PCA in the sense that it may provide a representation more robust to the effect of noise [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R42" rid="R42" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">42</a>]. It is, therefore, possible for ICA to be better than PCA for reconstruction in noisy or limited precision environments. For example, in the problem presented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, we found that if only 12 bits are allowed to represent the PCA and ICA coefficients, linear reconstructions based on ICA are 3 dB better than reconstructions based on PCA (the noise power is reduced by more than half). A similar result was obtained for PCA and ICA subspaces. If only four bits are allowed to represent the first 2 PCA and ICA coefficients, ICA reconstructions are 3 dB better than PCA reconstructions. In some problems, one can think of the actual inputs as noisy versions of some canonical inputs. For example, variations in lighting and expressions can be seen as noisy versions of the canonical image of a person. Having input representations which are robust to noise may potentially give us representations that better reflect the data.</p><p id="P16" class="p p-last">When the sources models are sparse, ICA is closely related to the so called nonorthogonal “rotation” methods in PCA and factor analysis. The goal of these rotation methods is to find directions with high concentrations of data, something very similar to what ICA does when the sources are sparse. In such cases, ICA can be seen as a theoretically sound probabilistic method to find interesting nonorthogonal “rotations.”</p></div><div id="S5" class="sec sec-last"><p></p><h4 class="inline">ICA and Cluster Analysis </h4><p id="P17" class="p p-first">Cluster analysis is a technique for finding regions in <em>n</em>-dimensional space with large concentrations of data. These regions are called “clusters.” Typically, the main statistic of interest in cluster analysis is the center of those clusters. When the source models are sparse, ICA finds directions along which significant concentrations of data points are observed. Thus, when using sparse sources, ICA can be seen as a form of cluster analysis. However, the emphasis in ICA is on finding optimal directions, rather than specific locations of high data density. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> illustrates this point. Note how the data concentrates along the ICA solutions, not the PCA solutions. Note also that in this case, all the clusters have equal mean, and thus are better characterized by their orientation rather than their position in space.</p><p id="P18" class="p p-last">It should be noted that ICA is a very general technique. When super-Gaussian sources are used, ICA can be seen as doing something akin to nonorthogonal PCA and to cluster analysis, however, when the source models are sub-Gaussian, the relationship between these techniques is less clear. See [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510574" role="button" aria-expanded="false" aria-haspopup="true">30</a>] for a discussion of ICA in the context of sub-Gaussian sources.</p></div></div><div id="S6" class="sec sec-last"><h3>B. Two Architectures for Performing ICA on Images</h3><p id="P19" class="p p-first">Let <em>X</em> be a data matrix with <em>n<sub>r</sub></em> rows and <em>n<sub>c</sub></em> columns. We can think of each column of <em>X</em> as outcomes (independent trials) of a random experiment. We think of the <em>i</em>th row of <em>X</em> as the specific value taken by a random variable <strong>X</strong><sub><em>i</em></sub> across <em>n<sub>c</sub></em> independent trials. This defines an empirical probability distribution for <strong>X</strong><sub>1</sub>, … <strong>X</strong><em><sub>nr</sub></em> in which each column of <em>X</em> is given probability mass 1/<em>n<sub>c</sub></em>. Independence is then defined with respect to such a distribution. For example, we say that rows <em>i</em> and <em>j</em> of <em>X</em> are independent if it is not possible to predict the values taken by <strong>X</strong><em><sub>j</sub></em> across columns from the corresponding values taken by <strong>X</strong><em><sub>j</sub></em>, i.e.,
</p><div class="disp-formula" id="FD7"><div class="f"><em>P</em>(<strong>X</strong><sub><em>i</em></sub> =&nbsp;<em>u</em>,&nbsp;<strong>X</strong><sub><em>j</em></sub> =&nbsp;<em>v</em>) =&nbsp;<em>P</em>(<strong>X</strong><sub><em>i</em></sub> =&nbsp;<em>u</em>)<em>P</em>(<strong>X</strong><sub><em>j</em></sub> =&nbsp;<em>v</em>) for all <em>u</em>,&nbsp;<em>v</em>&nbsp;∈&nbsp;<em>R</em></div><div class="l">(7)</div></div><p>
where <em>P</em> is the empirical distribution as in (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD7" rid="FD7" class=" disp-formula">7</a>).</p><p id="P20" class="p">Our goal in this paper is to find a good set of basis images to represent a database of faces. We organize each image in the database as a long vector with as many dimensions as number of pixels in the image. There are at least two ways in which ICA can be applied to this problem.</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=2
--><li class="a_label"><div class="inline_block a_label">1)</div><div>We can organize our database into a matrix <em>X</em> where each row vector is a different image. This approach is illustrated in (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> left). In this approach, images are random variables and pixels are trials. In this approach, it makes sense to talk about independence of images or functions of images. Two images <em>i</em> and <em>j</em> are independent if when moving across pixels, it is not possible to predict the value taken by the pixel on image <em>j</em> based on the value taken by the same pixel on image <em>i</em>. A similar approach was used by Bell and Sejnowski for sound source separation [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510578" role="button" aria-expanded="false" aria-haspopup="true">11</a>], for EEG analysis [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R37" rid="R37" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">37</a>], and for fMRI [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510558" role="button" aria-expanded="false" aria-haspopup="true">39</a>].</div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094238130704" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0006.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0006.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0006.jpg"></div></a><div id="largeobj_idm140094238130704" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a4--><div><span>Two architectures for performing ICA on images. (a) Architecture I for finding statistically independent basis images. Performing source separation on the face images produced IC images in the rows of <em>U</em>. (b) The gray values at pixel location <em>i</em> are plotted for each face image. ICA in architecture I finds weight vectors in the directions of statistical dependencies among the pixel locations. (c) Architecture II for finding a factorial code. Performing source separation on the pixels produced a factorial code in the columns of the output matrix, <em>U</em>. (d) Each face image is plotted according to the gray values taken on at each pixel location. ICA in architecture II finds weight vectors in the directions of statistical dependencies among the face images.</span></div></div></div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>We can transpose <em>X</em> and organize our data so that images are in the columns of <em>X</em>. This approach is illustrated in (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> right). In this approach, pixels are random variables and images are trials. Here, it makes sense to talk about independence of pixels or functions of pixels. For example, pixel <em>i</em> and <em>j</em> would be independent if when moving across the entire set of images it is not possible to predict the value taken by pixel <em>i</em> based on the corresponding value taken by pixel <em>j</em> on the same image. This approach was inspired by Bell and Sejnowski's work on the ICs of natural images [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510570" role="button" aria-expanded="false" aria-haspopup="true">12</a>].</div></li></ul></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">III. Image Data</h2><p id="P23" class="p p-first">The face images employed for this research were a subset of the FERET face database [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R52" rid="R52" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">52</a>]. The data set contained images of 425 individuals. There were up to four frontal views of each individual: A neutral expression and a change of expression from one session, and a neutral expression and change of expression from a second session that occurred up to two years after the first. Examples of the four views are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F6/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6</span></a>. The algorithms were trained on a single frontal view of each individual. The training set was comprised of 50% neutral expression images and 50% change of expression images. The algorithms were tested for recognition under three different conditions: same session, different expression; different day, same expression; and different day, different expression (see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/table/T1/" target="true" class="fig-table-link table figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094234901136" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0009.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0009.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0009.jpg"></div></a><div id="largeobj_idm140094234901136" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Fig. 6</a></div><!--caption a4--><div><span>Example from the FERET database of the four frontal image viewing conditions: neutral expression and change of expression from session 1; neutral expression and change of expression from session 2. Reprinted with permission from Jonathan Phillips.</span></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE I</h3><!--caption a7--><div class="caption"><p id="__p32">Image Sets Used for Training and Testing</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094234583328" class="xtable"><table frame="above" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="bottom" rowspan="1" colspan="1">Image Set</th><th colspan="2" align="left" valign="bottom" rowspan="1">Condition</th><th align="left" valign="bottom" rowspan="1" colspan="1">No. Images</th></tr></thead><tbody><tr><td align="left" valign="bottom" rowspan="1" colspan="1">Training Set</td><td align="left" valign="bottom" rowspan="1" colspan="1">Session I</td><td align="left" valign="bottom" rowspan="1" colspan="1">50% neutral 50% other</td><td align="left" valign="bottom" rowspan="1" colspan="1">425</td></tr><tr><td align="left" valign="bottom" rowspan="1" colspan="1">Test Set 1</td><td align="left" valign="bottom" rowspan="1" colspan="1">Same Day</td><td align="left" valign="bottom" rowspan="1" colspan="1">Different Expression</td><td align="left" valign="bottom" rowspan="1" colspan="1">421</td></tr><tr><td align="left" valign="bottom" rowspan="1" colspan="1">Test Set 2</td><td align="left" valign="bottom" rowspan="1" colspan="1">Different Day</td><td align="left" valign="bottom" rowspan="1" colspan="1">Same Expression</td><td align="left" valign="bottom" rowspan="1" colspan="1">45</td></tr><tr><td align="left" valign="bottom" rowspan="1" colspan="1">Test Set 3</td><td align="left" valign="bottom" rowspan="1" colspan="1">Different Day</td><td align="left" valign="bottom" rowspan="1" colspan="1">Different Expression</td><td align="left" valign="bottom" rowspan="1" colspan="1">43</td></tr></tbody></table></div><div id="largeobj_idm140094234583328" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P24" class="p p-last">Coordinates for eye and mouth locations were provided with the FERET database. These coordinates were used to center the face images, and then crop and scale them to 60 × 50 pixels. Scaling was based on the area of the triangle defined by the eyes and mouth. The luminance was normalized by linearly rescaling each image to the interval [0, 255]. For the subsequent analyses, each image was represented as a 3000–dimensional vector given by the luminance value at each pixel location.</p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">IV. Architecture I: Statistically Independent Basis Images</h2><p id="P25" class="p p-first">As described earlier, the goal in this approach is to find a set of statistically independent basis images. We organize the data matrix <em>X</em> so that the images are in rows and the pixels are in columns, i.e., <em>X</em> has 425 rows and 3000 columns, and each image has zero mean.</p><p id="P26">In this approach, ICA finds a matrix <em>W</em> such that the rows of <em>U</em> = <em>WX</em> are as statistically independent as possible. The source images estimated by the rows of <em>U</em> are then used as basis images to represent faces. Face image representations consist of the coordinates of these images with respect to the image basis defined by the rows of <em>U</em>, as shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F7/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Fig. 7</span></a>. These coordinates are contained in the mixing matrix <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M10" overflow="scroll" role="math" style="width: 4.825em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.792em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.036em, 1003.79em, 2.808em, -999.998em); top: -2.36em; left: 0.002em;"><span class="mrow" id="MathJax-Span-70"><span class="mrow" id="MathJax-Span-71"><span class="mi" id="MathJax-Span-72" style="font-family: STIXGeneral; font-style: italic;">A</span><span class="mover" id="MathJax-Span-73" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 0.691em; height: 0px;"><span style="position: absolute; clip: rect(3.447em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-74" style="font-family: STIXGeneral;">=</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.25em, 1000.4em, 4.136em, -999.998em); top: -4.574em; left: 0.15em;"><span class="mi" id="MathJax-Span-75" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">Δ</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-76" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-77" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-78"><span class="mo" id="MathJax-Span-79" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-80" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-81" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.365em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 2.003em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M10" overflow="scroll"><mrow><mi>A</mi><mover><mo>=</mo><mi>Δ</mi></mover><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094272870064" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0010.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0010.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0010.jpg"></div></a><div id="largeobj_idm140094272870064" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Fig. 7</a></div><!--caption a4--><div><span>The independent basis image representation consisted of the coefficients, <strong>b</strong>, for the linear combination of independent basis images, <strong>u</strong>, that comprised each face image <strong>x</strong>.</span></div></div></div><p id="P27">The number of ICs found by the ICA algorithm corresponds to the dimensionality of the input. Since we had 425 images in the training set, the algorithm would attempt to separate 425 ICs. Although we found in previous work that performance improved with the number of components separated, 425 was intractable under our present memory limitations. In order to have control over the number of ICs extracted by the algorithm, instead of performing ICA on the <em>n<sub>r</sub></em> original images, we performed ICA on a set of <em>m</em> linear combinations of those images, where <em>m</em> &lt; <em>n<sub>r</sub></em>. Recall that the image synthesis model assumes that the images in <em>X</em> are a linear combination of a set of unknown statistically independent sources. The image synthesis model is unaffected by replacing the original images with some other linear combination of the images.</p><p id="P28">Adopting a method that has been applied to ICA of fMRI data [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510567" role="button" aria-expanded="false" aria-haspopup="true">39</a>], we chose for these linear combinations the first <em>m</em> PC eigenvectors of the image set. PCA on the image set in which the pixel locations are treated as observations and each face image a measure, gives the linear combination of the parameters (images) that accounts for the maximum variability in the observations (pixels). The use of PCA vectors in the input did not throw away the high-order relationships. These relationships still existed in the data but were not separated.</p><p id="P29">Let <em>P<sub>m</sub></em> denote the matrix containing the first <em>m</em> PC axes in its columns. We performed ICA on <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M11" overflow="scroll" role="math" style="width: 1.528em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.052em, 1001.18em, 1.282em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-83"><span class="mrow" id="MathJax-Span-84"><span class="msubsup" id="MathJax-Span-85"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-86" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-87" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.546em, 1000.59em, 4.136em, -999.998em); top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-88" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.247em; border-left: 0px solid; width: 0px; height: 1.378em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M11" overflow="scroll"><mrow><msubsup><mi>P</mi><mi>m</mi><mi>T</mi></msubsup></mrow></math></script>, producing a matrix of <em>m</em> independent source images in the rows of <em>U</em>. In this implementation, the coefficients <strong>b</strong> for the linear combination of basis images in <em>U</em> that comprised the face images in <em>X</em> were determined as follows.</p><p id="P30">The PC representation of the set of zero-mean images in <em>X</em> based on <em>P<sub>m</sub></em> is defined as <em>R<sub>m</sub></em> = <em>X P<sub>m</sub></em>. A minimum squared error approximation of <em>X</em> is obtained by <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M12" overflow="scroll" role="math" style="width: 5.957em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.678em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.331em, 1004.68em, 2.758em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-90"><span class="mrow" id="MathJax-Span-91"><span class="mover" id="MathJax-Span-92"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.15em;"><span class="mi" id="MathJax-Span-93" style="font-family: STIXGeneral; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1000.99em, 3.595em, -999.998em); top: -4.181em; left: 0.052em;"><span class="mo" id="MathJax-Span-94" style=""><span style="font-family: STIXSizeTwoSym;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-95" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msub" id="MathJax-Span-96" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-97" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-98" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-99"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-100" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-101" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.546em, 1000.59em, 4.136em, -999.998em); top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.247em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M12" overflow="scroll"><mrow><mover accent="true"><mi>X</mi><mo>^</mo></mover><mo>=</mo><msub><mi>R</mi><mi>m</mi></msub><msubsup><mi>P</mi><mi>m</mi><mi>T</mi></msubsup></mrow></math></script>.</p><p id="P31">The ICA algorithm produced a matrix <em>W<sub>I</sub></em> = <em>WW<sub>Z</sub></em> such that
</p><div class="disp-formula" id="FD8"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0" style=""><nobr><span class="math" id="M13" overflow="scroll" role="math" style="width: 9.5em; display: inline-block;"><span style="display: inline-block; position: relative; width: 7.483em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.396em, 1007.29em, 3.3em, -999.998em); top: -2.114em; left: 0.002em;"><span class="mrow" id="MathJax-Span-104"><span class="mtable" id="MathJax-Span-105" style="padding-right: 0.15em; padding-left: 0.15em;"><span style="display: inline-block; position: relative; width: 7.188em; height: 0px;"><span style="position: absolute; clip: rect(2.266em, 1002.37em, 5.071em, -999.998em); top: -3.984em; left: 0.002em;"><span style="display: inline-block; position: relative; width: 2.365em; height: 0px;"><span style="position: absolute; clip: rect(3.054em, 1002.37em, 4.284em, -999.998em); top: -4.722em; right: 0.002em;"><span class="mtd" id="MathJax-Span-106"><span class="mrow" id="MathJax-Span-107"><span class="mrow" id="MathJax-Span-108"><span class="msub" id="MathJax-Span-109"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-110" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.839em;"><span class="mi" id="MathJax-Span-111" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-112"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-113" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-114" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.546em, 1000.59em, 4.136em, -999.998em); top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-115" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1001.18em, 4.284em, -999.998em); top: -3.246em; right: 0.002em;"><span class="mtd" id="MathJax-Span-121"><span class="mrow" id="MathJax-Span-122"><span class="mrow" id="MathJax-Span-123"><span class="msubsup" id="MathJax-Span-124"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-125" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-126" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.546em, 1000.59em, 4.136em, -999.998em); top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-127" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(2.463em, 1003.84em, 5.17em, -999.998em); top: -3.984em; left: 3.3em;"><span style="display: inline-block; position: relative; width: 3.89em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1001.77em, 4.136em, -999.998em); top: -4.722em; left: 0.002em;"><span class="mtd" id="MathJax-Span-116"><span class="mrow" id="MathJax-Span-117"><span class="mrow" id="MathJax-Span-118"><span class="mo" id="MathJax-Span-119" style="font-family: STIXGeneral;">=</span><span class="mi" id="MathJax-Span-120" style="font-family: STIXGeneral; font-style: italic; padding-left: 0.298em;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.004em, 1003.84em, 4.432em, -999.998em); top: -3.246em; left: 0.002em;"><span class="mtd" id="MathJax-Span-128"><span class="mrow" id="MathJax-Span-129"><span class="mrow" id="MathJax-Span-130"><span class="mo" id="MathJax-Span-131" style="font-family: STIXGeneral;">=</span><span class="msubsup" id="MathJax-Span-132" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-133" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-134"><span class="mo" id="MathJax-Span-135" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-136" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-137" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mi" id="MathJax-Span-138" style="font-family: STIXGeneral; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span class="mo" id="MathJax-Span-139" style="font-family: STIXGeneral;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.119em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.372em; border-left: 0px solid; width: 0px; height: 3.441em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M13" overflow="scroll"><mtable><mtr><mtd columnalign="right"><mrow><msub><mi>W</mi><mi>I</mi></msub><msubsup><mi>P</mi><mi>m</mi><mi>T</mi></msubsup></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><mi>U</mi></mrow></mtd></mtr><mtr><mtd columnalign="right"><mrow><msubsup><mi>P</mi><mi>m</mi><mi>T</mi></msubsup></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mi>U</mi><mo>.</mo></mrow></mtd></mtr></mtable></math></script></div><div class="l">(8)</div></div><p>
Therefore
</p><div class="disp-formula" id="FD9"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0" style=""><nobr><span class="math" id="M14" overflow="scroll" role="math" style="width: 9.353em; display: inline-block;"><span style="display: inline-block; position: relative; width: 7.335em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.249em, 1007.14em, 3.497em, -999.998em); top: -2.114em; left: 0.002em;"><span class="mrow" id="MathJax-Span-141"><span class="mtable" id="MathJax-Span-142" style="padding-right: 0.15em; padding-left: 0.15em;"><span style="display: inline-block; position: relative; width: 7.04em; height: 0px;"><span style="position: absolute; clip: rect(2.119em, 1001.04em, 5.071em, -999.998em); top: -3.984em; left: 0.002em;"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(2.857em, 1001.04em, 4.136em, -999.998em); top: -4.722em; right: 0.002em;"><span class="mtd" id="MathJax-Span-143"><span class="mrow" id="MathJax-Span-144"><span class="mover" id="MathJax-Span-145"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.15em;"><span class="mi" id="MathJax-Span-146" style="font-family: STIXGeneral; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1000.99em, 3.595em, -999.998em); top: -4.181em; left: 0.052em;"><span class="mo" id="MathJax-Span-147" style=""><span style="font-family: STIXSizeTwoSym;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(2.857em, 1001.04em, 4.136em, -999.998em); top: -3.049em; right: 0.002em;"><span class="mtd" id="MathJax-Span-159"><span class="mrow" id="MathJax-Span-160"><span class="mover" id="MathJax-Span-161"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.15em;"><span class="mi" id="MathJax-Span-162" style="font-family: STIXGeneral; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1000.99em, 3.595em, -999.998em); top: -4.181em; left: 0.052em;"><span class="mo" id="MathJax-Span-163" style=""><span style="font-family: STIXSizeTwoSym;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(2.315em, 1005.02em, 5.367em, -999.998em); top: -3.984em; left: 1.922em;"><span style="display: inline-block; position: relative; width: 5.071em; height: 0px;"><span style="position: absolute; clip: rect(3.054em, 1003.4em, 4.284em, -999.998em); top: -4.722em; left: 0.002em;"><span class="mtd" id="MathJax-Span-148"><span class="mrow" id="MathJax-Span-149"><span class="mrow" id="MathJax-Span-150"><span class="mo" id="MathJax-Span-151" style="font-family: STIXGeneral;">=</span><span class="msub" id="MathJax-Span-152" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-153" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-154" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-155"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-156" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-157" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.546em, 1000.59em, 4.136em, -999.998em); top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-158" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.004em, 1005.02em, 4.432em, -999.998em); top: -3.049em; left: 0.002em;"><span class="mtd" id="MathJax-Span-164"><span class="mrow" id="MathJax-Span-165"><span class="mrow" id="MathJax-Span-166"><span class="mo" id="MathJax-Span-167" style="font-family: STIXGeneral;">=</span><span class="msub" id="MathJax-Span-168" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-169" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-170" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-171"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-172" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-173"><span class="mo" id="MathJax-Span-174" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-175" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-176" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mi" id="MathJax-Span-177" style="font-family: STIXGeneral; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span class="mo" id="MathJax-Span-178" style="font-family: STIXGeneral;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.119em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.622em; border-left: 0px solid; width: 0px; height: 3.878em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M14" overflow="scroll"><mtable><mtr><mtd columnalign="right"><mover accent="true"><mi>X</mi><mo>^</mo></mover></mtd><mtd columnalign="left"><mrow><mo>=</mo><msub><mi>R</mi><mi>m</mi></msub><msubsup><mi>P</mi><mi>m</mi><mi>T</mi></msubsup></mrow></mtd></mtr><mtr><mtd columnalign="right"><mover accent="true"><mi>X</mi><mo>^</mo></mover></mtd><mtd columnalign="left"><mrow><mo>=</mo><msub><mi>R</mi><mi>m</mi></msub><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mi>U</mi><mo>.</mo></mrow></mtd></mtr></mtable></math></script></div><div class="l">(9)</div></div><p>
where <em>W<sub>Z</sub></em> was the sphering matrix defined in (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD4" rid="FD4" class=" disp-formula">4</a>). Hence, the rows of <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M15" overflow="scroll" role="math" style="width: 3.89em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.054em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.002em, 1003.05em, 1.43em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-180"><span class="mrow" id="MathJax-Span-181"><span class="msub" id="MathJax-Span-182"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-183" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-184" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-185"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-186" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-187"><span class="mo" id="MathJax-Span-188" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-189" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-190" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M15" overflow="scroll"><mrow><msub><mi>R</mi><mi>m</mi></msub><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script> contained the coefficients for the linear combination of statistically independent sources <em>U</em> that comprised <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M16" overflow="scroll" role="math" style="width: 1.331em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(-0.145em, 1001.04em, 1.134em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-192"><span class="mover" id="MathJax-Span-193"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.15em;"><span class="mi" id="MathJax-Span-194" style="font-family: STIXGeneral; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1000.99em, 3.595em, -999.998em); top: -4.181em; left: 0.052em;"><span class="mo" id="MathJax-Span-195" style=""><span style="font-family: STIXSizeTwoSym;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.378em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M16" overflow="scroll"><mover accent="true"><mi>X</mi><mo>^</mo></mover></math></script>, where <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-11-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M17" overflow="scroll" role="math" style="width: 1.331em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(-0.145em, 1001.04em, 1.134em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-197"><span class="mover" id="MathJax-Span-198"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.15em;"><span class="mi" id="MathJax-Span-199" style="font-family: STIXGeneral; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.054em, 1000.99em, 3.595em, -999.998em); top: -4.181em; left: 0.052em;"><span class="mo" id="MathJax-Span-200" style=""><span style="font-family: STIXSizeTwoSym;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.378em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-11"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M17" overflow="scroll"><mover accent="true"><mi>X</mi><mo>^</mo></mover></math></script> was a minimum squared error approximation of <em>X</em>, just as in PCA. The IC representation of the face images based on the set of <em>m</em> statistically independent feature images, <em>U</em> was, therefore, given by the rows of the matrix
</p><div class="disp-formula" id="FD10"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-12-Frame" tabindex="0" style=""><nobr><span class="math" id="M18" overflow="scroll" role="math" style="width: 6.597em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.17em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.479em, 1005.12em, 2.906em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-202"><span class="mrow" id="MathJax-Span-203"><span class="mi" id="MathJax-Span-204" style="font-family: STIXGeneral; font-style: italic;">B</span><span class="mo" id="MathJax-Span-205" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msub" id="MathJax-Span-206" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-207" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mi" id="MathJax-Span-208" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">m</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-209"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-210" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-211"><span class="mo" id="MathJax-Span-212" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-213" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-214" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-215" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-12"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M18" overflow="scroll"><mrow><mi>B</mi><mo>=</mo><msub><mi>R</mi><mi>m</mi></msub><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo>.</mo></mrow></math></script></div><div class="l">(10)</div></div><p>
A representation for test images was obtained by using the PC representation based on the training images to obtain <em>R</em><sub>test</sub> = <em>X</em><sub>test</sub><em>P<sub>m</sub></em>, and then computing
</p><div class="disp-formula" id="FD11"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-13-Frame" tabindex="0" style=""><nobr><span class="math" id="M19" overflow="scroll" role="math" style="width: 8.565em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.745em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.479em, 1006.7em, 2.906em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-217"><span class="mrow" id="MathJax-Span-218"><span class="msub" id="MathJax-Span-219"><span style="display: inline-block; position: relative; width: 1.676em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-220" style="font-family: STIXGeneral; font-style: italic;">B</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mtext" id="MathJax-Span-221" style="font-size: 70.7%; font-family: STIXGeneral;">test</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-222" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msub" id="MathJax-Span-223" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.676em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-224" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.593em;"><span class="mtext" id="MathJax-Span-225" style="font-size: 70.7%; font-family: STIXGeneral;">test</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-226"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-227" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-228"><span class="mo" id="MathJax-Span-229" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-230" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-231" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-232" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-13"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M19" overflow="scroll"><mrow><msub><mi>B</mi><mtext>test</mtext></msub><mo>=</mo><msub><mi>R</mi><mtext>test</mtext></msub><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup><mo>.</mo></mrow></math></script></div><div class="l">(11)</div></div><p>
Note that the PCA step is not required for the ICA representation of faces. It was employed to serve two purposes: 1) to reduce the number of sources to a tractable number and 2) to provide a convenient method for calculating representations of test images. Without the PCA step, <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-14-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M20" overflow="scroll" role="math" style="width: 4.776em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.743em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.479em, 1003.74em, 2.906em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-234"><span class="mrow" id="MathJax-Span-235"><span class="mi" id="MathJax-Span-236" style="font-family: STIXGeneral; font-style: italic;">B</span><span class="mo" id="MathJax-Span-237" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msubsup" id="MathJax-Span-238" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-239" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-240"><span class="mo" id="MathJax-Span-241" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-242" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-243" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-14"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M20" overflow="scroll"><mrow><mi>B</mi><mo>=</mo><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script> and <em>B</em><sub>test</sub> = <em>X</em><sub>test</sub>(<em>U</em>)<sup>†</sup>.<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN4" rid="FN4" class=" fn">3</a></sup></p><p id="P32">The PC axes of the training set were found by calculating the eigenvectors of the pixelwise covariance matrix over the set of face images. ICA was then performed on the first <em>m</em> = 200 of these eigenvectors, where the first 200 PCs accounted for over 98% of the variance in the images.<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN5" rid="FN5" class=" fn">4</a></sup> The 1 × 3000 eigenvectors in <em>P</em><sub>200</sub> comprised the rows of the 200 × 3000 input matrix <em>X</em>. The input matrix <em>X</em> was sphered<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN6" rid="FN6" class=" fn">5</a></sup> according to (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD4" rid="FD4" class=" disp-formula">4</a>), and the weights <em>W</em> were updated according to (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD3" rid="FD3" class=" disp-formula">3</a>) for 1900 iterations. The learning rate was initialized at 0.0005 and annealed down to 0.0001. Training took 90 minutes on a Dec Alpha 2100a. Following training, a set of statistically independent source images were contained in the rows of the output matrix <em>U</em>.</p><p id="P33" class="p"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Fig. 8</span></a> shows a subset of 25 basis images (i.e., rows of <em>U</em>). These images can be interpreted as follows: Each row of the mixing matrix <em>W</em> found by ICA represents a cluster of pixels that have similar behavior across images. Each row of the <em>U</em> matrix tell us how close each pixel is to the cluster <em>i</em> identified by ICA. Since we use a sparse independent source model, these basis images are expected to be sparse and independent. Sparseness in this case means that the basis images will have a large number of pixels close to zero and a few pixels with large positive or negative values. Note that the ICA images are also local (regions with nonzero pixels are nearby). This is because a majority of the statistical dependencies are in spatially proximal pixel locations. A set of PC basis images (PCA axes) are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F9/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Fig. 9</span></a> for comparison.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F8" co-legend-rid="lgnd_F8"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094229364016" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0011.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0011.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0011.jpg"></a></div><div id="largeobj_idm140094229364016" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Fig. 8</a></div><!--caption a4--><div><span>Twenty-five ICs of the image set obtained by Architecture I, which provide a set of statistically independent basis images (rows of <em>U</em> in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). ICs are ordered by the class discriminability ratio, <em>r</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD4" rid="FD4" class=" disp-formula">4</a>).</span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F9" co-legend-rid="lgnd_F9"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094230509808" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0012.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0012.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0012.jpg"></a></div><div id="largeobj_idm140094230509808" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F9/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Fig. 9</a></div><!--caption a4--><div><span>First 25 PC axes of the image set (columns of <em>P</em>), ordered left to right, top to bottom, by the magnitude of the corresponding eigenvalue.</span></div></div></div><div id="S9" class="sec"><h3>A. Face Recognition Performance: Architecture I</h3><p id="P34" class="p p-first">Face recognition performance was evaluated for the coefficient vectors <strong>b</strong> by the nearest neighbor algorithm, using cosines as the similarity measure. Coefficient vectors in each test set were assigned the class label of the coefficient vector in the training set that was most similar as evaluated by the cosine of the angle between them
</p><div class="disp-formula" id="FD12"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-15-Frame" tabindex="0" style=""><nobr><span class="math" id="M21" overflow="scroll" role="math" style="width: 10.337em; display: inline-block;"><span style="display: inline-block; position: relative; width: 8.123em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.987em, 1008.07em, 3.497em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-245"><span class="mrow" id="MathJax-Span-246"><span class="mi" id="MathJax-Span-247" style="font-family: STIXGeneral; font-style: italic;">c</span><span class="mo" id="MathJax-Span-248" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mfrac" id="MathJax-Span-249" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 5.957em; height: 0px; margin-right: 0.101em; margin-left: 0.101em;"><span style="position: absolute; clip: rect(3.152em, 1004.33em, 4.284em, -999.998em); top: -4.673em; left: 50%; margin-left: -2.163em;"><span class="mrow" id="MathJax-Span-250"><span class="msub" id="MathJax-Span-251"><span style="display: inline-block; position: relative; width: 1.626em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-252" style="font-family: STIXGeneral; font-weight: bold;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mtext" id="MathJax-Span-253" style="font-size: 70.7%; font-family: STIXGeneral;">test</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-254" style="font-family: STIXGeneral; padding-left: 0.249em;">⋅</span><span class="msub" id="MathJax-Span-255" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 1.922em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-256" style="font-family: STIXGeneral; font-weight: bold;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mtext" id="MathJax-Span-257" style="font-size: 70.7%; font-family: STIXGeneral;">train</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.152em, 1005.71em, 4.333em, -999.998em); top: -3.295em; left: 50%; margin-left: -2.901em;"><span class="mrow" id="MathJax-Span-258"><span class="mrow" id="MathJax-Span-259"><span class="mo" id="MathJax-Span-260" style="font-family: STIXGeneral;">‖</span><span class="msub" id="MathJax-Span-261"><span style="display: inline-block; position: relative; width: 1.626em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-262" style="font-family: STIXGeneral; font-weight: bold;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mtext" id="MathJax-Span-263" style="font-size: 70.7%; font-family: STIXGeneral;">test</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-264" style="font-family: STIXGeneral;">‖</span></span><span class="mrow" id="MathJax-Span-265" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-266" style="font-family: STIXGeneral;">‖</span><span class="msub" id="MathJax-Span-267"><span style="display: inline-block; position: relative; width: 1.922em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-268" style="font-family: STIXGeneral; font-weight: bold;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mtext" id="MathJax-Span-269" style="font-size: 70.7%; font-family: STIXGeneral;">train</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-270" style="font-family: STIXGeneral;">‖</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(0.888em, 1005.96em, 1.233em, -999.998em); top: -1.277em; left: 0.002em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.002em; border-top: 1.3px solid; width: 5.957em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-271" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.184em; border-left: 0px solid; width: 0px; height: 2.941em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-15"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M21" overflow="scroll"><mrow><mi>c</mi><mo>=</mo><mfrac><mrow><msub><mi mathvariant="bold">b</mi><mtext>test</mtext></msub><mo>⋅</mo><msub><mi mathvariant="bold">b</mi><mtext>train</mtext></msub></mrow><mrow><mrow><mo>‖</mo><msub><mi mathvariant="bold">b</mi><mtext>test</mtext></msub><mo>‖</mo></mrow><mrow><mo>‖</mo><msub><mi mathvariant="bold">b</mi><mtext>train</mtext></msub><mo>‖</mo></mrow></mrow></mfrac><mo>.</mo></mrow></math></script></div><div class="l">(12)</div></div><p>
Face recognition performance for the PC representation was evaluated by an identical procedure, using the PC coefficients contained in the rows of <em>R</em><sub>200</sub>.</p><p id="P35">In experiments to date, ICA performs significantly better using cosines rather than Euclidean distance as the similarity measure, whereas PCA performs the same for both. A cosine similarity measure is equivalent to length-normalizing the vectors prior to measuring Euclidean distance when doing nearest neighbor
</p><div class="disp-formula" id="FD13"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-16-Frame" tabindex="0" style=""><nobr><span class="math" id="M22" overflow="scroll" role="math" style="width: 27.463em; display: inline-block;"><span style="display: inline-block; position: relative; width: 21.607em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(-1.179em, 1021.41em, 4.924em, -999.998em); top: -2.114em; left: 0.002em;"><span class="mrow" id="MathJax-Span-273"><span class="mtable" id="MathJax-Span-274" style="padding-right: 0.15em; padding-left: 0.15em;"><span style="display: inline-block; position: relative; width: 21.312em; height: 0px;"><span style="position: absolute; clip: rect(3.497em, 1005.66em, 9.55em, -999.998em); top: -6.74em; left: 0.002em;"><span style="display: inline-block; position: relative; width: 5.711em; height: 0px;"><span style="position: absolute; clip: rect(3.004em, 1003.15em, 4.333em, -999.998em); top: -6.248em; right: 0.002em;"><span class="mtd" id="MathJax-Span-275"><span class="mrow" id="MathJax-Span-276"><span class="mrow" id="MathJax-Span-277"><span class="msup" id="MathJax-Span-278"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-279" style="font-family: STIXGeneral; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.328em; left: 0.593em;"><span class="mn" id="MathJax-Span-280" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-281" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-282" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">(</span></span><span class="mi" id="MathJax-Span-283" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span class="mo" id="MathJax-Span-284" style="font-family: STIXGeneral;">,</span><span class="mi" id="MathJax-Span-285" style="font-family: STIXGeneral; font-style: italic; padding-left: 0.199em;">y</span><span class="mo" id="MathJax-Span-286" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">)</span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.841em, 1000em, 4.136em, -999.998em); top: -4.673em; right: 0.002em;"><span class="mtd" id="MathJax-Span-311"><span class="mrow" id="MathJax-Span-312"><span class="mrow" id="MathJax-Span-313"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.152em, 1005.02em, 4.333em, -999.998em); top: -3.147em; right: 0.002em;"><span class="mtd" id="MathJax-Span-349"><span class="mrow" id="MathJax-Span-350"><span class="mrow" id="MathJax-Span-351"><span class="mtext" id="MathJax-Span-352" style="font-family: STIXGeneral;">Thus, if</span><span class="mspace" id="MathJax-Span-353" style="height: 0.002em; vertical-align: 0.002em; width: 0.298em; display: inline-block; overflow: hidden;"></span><span class="mrow" id="MathJax-Span-354" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-355" style="font-family: STIXGeneral;">‖</span><span class="mi" id="MathJax-Span-356" style="font-family: STIXGeneral; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-357" style="font-family: STIXGeneral;">‖</span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.004em, 1005.66em, 4.432em, -999.998em); top: -1.622em; right: 0.002em;"><span class="mtd" id="MathJax-Span-368"><span class="mrow" id="MathJax-Span-369"><span class="mrow" id="MathJax-Span-370"><span class="msub" id="MathJax-Span-371"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1001.53em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-372" style="font-family: STIXGeneral;">min</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 1.577em;"><span class="mi" id="MathJax-Span-373" style="font-size: 70.7%; font-family: STIXGeneral; font-weight: bold;">y</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mspace" id="MathJax-Span-374" style="height: 0.002em; vertical-align: 0.002em; width: 0.298em; display: inline-block; overflow: hidden;"></span><span class="msup" id="MathJax-Span-375" style="padding-left: 0.199em;"><span style="display: inline-block; position: relative; width: 1.036em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-376" style="font-family: STIXGeneral; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.328em; left: 0.593em;"><span class="mn" id="MathJax-Span-377" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-378" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-379" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">(</span></span><span class="mi" id="MathJax-Span-380" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span class="mo" id="MathJax-Span-381" style="font-family: STIXGeneral;">,</span><span class="mi" id="MathJax-Span-382" style="font-family: STIXGeneral; font-style: italic; padding-left: 0.199em;">y</span><span class="mo" id="MathJax-Span-383" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">)</span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 6.745em;"></span></span><span style="position: absolute; clip: rect(3.497em, 1014.62em, 9.599em, -999.998em); top: -6.789em; left: 6.597em;"><span style="display: inline-block; position: relative; width: 14.668em; height: 0px;"><span style="position: absolute; clip: rect(2.955em, 1009.55em, 4.333em, -999.998em); top: -6.248em; left: 0.002em;"><span class="mtd" id="MathJax-Span-287"><span class="mrow" id="MathJax-Span-288"><span class="mrow" id="MathJax-Span-289"><span class="mo" id="MathJax-Span-290" style="font-family: STIXGeneral;">=</span><span class="mrow" id="MathJax-Span-291" style="padding-left: 0.298em;"><span class="msup" id="MathJax-Span-292"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1001.43em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-293"><span class="mo" id="MathJax-Span-294" style="font-family: STIXGeneral;">‖</span><span class="mi" id="MathJax-Span-295" style="font-family: STIXGeneral; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-296" style="font-family: STIXGeneral;">‖</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.377em; left: 1.528em;"><span class="mn" id="MathJax-Span-297" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-298" style="font-family: STIXGeneral; padding-left: 0.249em;">+</span><span class="mrow" id="MathJax-Span-299" style="padding-left: 0.249em;"><span class="msup" id="MathJax-Span-300"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1001.43em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-301"><span class="mo" id="MathJax-Span-302" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span><span class="mi" id="MathJax-Span-303" style="font-family: STIXGeneral; font-weight: bold;">y</span><span class="mo" id="MathJax-Span-304" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.377em; left: 1.528em;"><span class="mn" id="MathJax-Span-305" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-306" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mn" id="MathJax-Span-307" style="font-family: STIXGeneral; padding-left: 0.249em;">2</span><span class="mi" id="MathJax-Span-308" style="font-family: STIXGeneral; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-309" style="font-family: STIXGeneral; padding-left: 0.249em;">⋅</span><span class="mi" id="MathJax-Span-310" style="font-family: STIXGeneral; font-weight: bold; padding-left: 0.249em;">y</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(2.955em, 1014.62em, 4.333em, -999.998em); top: -4.673em; left: 0.002em;"><span class="mtd" id="MathJax-Span-314"><span class="mrow" id="MathJax-Span-315"><span class="mrow" id="MathJax-Span-316"><span class="mo" id="MathJax-Span-317" style="font-family: STIXGeneral;">=</span><span class="mrow" id="MathJax-Span-318" style="padding-left: 0.298em;"><span class="msup" id="MathJax-Span-319"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1001.43em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-320"><span class="mo" id="MathJax-Span-321" style="font-family: STIXGeneral;">‖</span><span class="mi" id="MathJax-Span-322" style="font-family: STIXGeneral; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-323" style="font-family: STIXGeneral;">‖</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.377em; left: 1.528em;"><span class="mn" id="MathJax-Span-324" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-325" style="font-family: STIXGeneral; padding-left: 0.249em;">+</span><span class="mrow" id="MathJax-Span-326" style="padding-left: 0.249em;"><span class="msup" id="MathJax-Span-327"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1001.43em, 4.333em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-328"><span class="mo" id="MathJax-Span-329" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span><span class="mi" id="MathJax-Span-330" style="font-family: STIXGeneral; font-weight: bold;">y</span><span class="mo" id="MathJax-Span-331" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.377em; left: 1.528em;"><span class="mn" id="MathJax-Span-332" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-333" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mn" id="MathJax-Span-334" style="font-family: STIXGeneral; padding-left: 0.249em;">2</span><span class="mrow" id="MathJax-Span-335" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-336" style="font-family: STIXGeneral;">‖</span><span class="mi" id="MathJax-Span-337" style="font-family: STIXGeneral; font-weight: bold;">x</span><span class="mo" id="MathJax-Span-338" style="font-family: STIXGeneral;">‖</span></span><span class="mrow" id="MathJax-Span-339" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-340" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span><span class="mi" id="MathJax-Span-341" style="font-family: STIXGeneral; font-weight: bold;">y</span><span class="mo" id="MathJax-Span-342" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span></span><span class="mi" id="MathJax-Span-343" style="font-family: STIXGeneral; padding-left: 0.199em;">cos</span><span class="mrow" id="MathJax-Span-344" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-345" style="font-family: STIXGeneral;">(</span><span class="mi" id="MathJax-Span-346" style="font-family: STIXGeneral; font-style: italic;">α</span><span class="mo" id="MathJax-Span-347" style="font-family: STIXGeneral;">)</span></span><span class="mo" id="MathJax-Span-348" style="font-family: STIXGeneral; padding-left: 0.199em;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.152em, 1004.19em, 4.333em, -999.998em); top: -3.147em; left: 0.002em;"><span class="mtd" id="MathJax-Span-358"><span class="mrow" id="MathJax-Span-359"><span class="mrow" id="MathJax-Span-360"><span class="mo" id="MathJax-Span-361" style="font-family: STIXGeneral;">=</span><span class="mrow" id="MathJax-Span-362" style="padding-left: 0.298em;"><span class="mo" id="MathJax-Span-363" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span><span class="mi" id="MathJax-Span-364" style="font-family: STIXGeneral; font-weight: bold;">y</span><span class="mo" id="MathJax-Span-365" style="vertical-align: 0.002em;"><span style="font-family: STIXGeneral;">‖</span></span></span><span class="mo" id="MathJax-Span-366" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mn" id="MathJax-Span-367" style="font-family: STIXGeneral; padding-left: 0.298em;">1</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.152em, 1006.79em, 4.432em, -999.998em); top: -1.622em; left: 0.002em;"><span class="mtd" id="MathJax-Span-384"><span class="mrow" id="MathJax-Span-385"><span class="mrow" id="MathJax-Span-386"><span class="mo" id="MathJax-Span-387" style="font-family: STIXGeneral;">=</span><span class="msub" id="MathJax-Span-388" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1001.73em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-389" style="font-family: STIXGeneral;">max</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 1.725em;"><span class="mi" id="MathJax-Span-390" style="font-size: 70.7%; font-family: STIXGeneral; font-weight: bold;">y</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mspace" id="MathJax-Span-391" style="height: 0.002em; vertical-align: 0.002em; width: 0.298em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-392" style="font-family: STIXGeneral; padding-left: 0.199em;">cos</span><span class="mrow" id="MathJax-Span-393" style="padding-left: 0.199em;"><span class="mo" id="MathJax-Span-394" style="font-family: STIXGeneral;">(</span><span class="mi" id="MathJax-Span-395" style="font-family: STIXGeneral; font-style: italic;">α</span><span class="mo" id="MathJax-Span-396" style="font-family: STIXGeneral;">)</span></span><span class="mo" id="MathJax-Span-397" style="font-family: STIXGeneral; padding-left: 0.199em;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span><span style="display: inline-block; width: 0px; height: 6.794em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.119em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -3.434em; border-left: 0px solid; width: 0px; height: 7.503em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-16"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M22" overflow="scroll"><mtable><mtr><mtd columnalign="right"><mrow><msup><mi>d</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo>)</mo></mrow></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><mrow><msup><mrow><mo>‖</mo><mi mathvariant="bold">x</mi><mo>‖</mo></mrow><mn>2</mn></msup></mrow><mo>+</mo><mrow><msup><mrow><mo>‖</mo><mi mathvariant="bold">y</mi><mo>‖</mo></mrow><mn>2</mn></msup></mrow><mo>−</mo><mn>2</mn><mi mathvariant="bold">x</mi><mo>⋅</mo><mi mathvariant="bold">y</mi></mrow></mtd></mtr><mtr><mtd columnalign="right"><mrow></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><mrow><msup><mrow><mo>‖</mo><mi mathvariant="bold">x</mi><mo>‖</mo></mrow><mn>2</mn></msup></mrow><mo>+</mo><mrow><msup><mrow><mo>‖</mo><mi mathvariant="bold">y</mi><mo>‖</mo></mrow><mn>2</mn></msup></mrow><mo>−</mo><mn>2</mn><mrow><mo>‖</mo><mi mathvariant="bold">x</mi><mo>‖</mo></mrow><mrow><mo>‖</mo><mi mathvariant="bold">y</mi><mo>‖</mo></mrow><mi>cos</mi><mrow><mo>(</mo><mi>α</mi><mo>)</mo></mrow><mo>.</mo></mrow></mtd></mtr><mtr><mtd columnalign="right"><mrow><mtext>Thus, if</mtext><mspace width="thickmathspace"></mspace><mrow><mo>‖</mo><mi mathvariant="bold">x</mi><mo>‖</mo></mrow></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><mrow><mo>‖</mo><mi mathvariant="bold">y</mi><mo>‖</mo></mrow><mo>=</mo><mn>1</mn></mrow></mtd></mtr><mtr><mtd columnalign="right"><mrow><msub><mi>min</mi><mi mathvariant="bold">y</mi></msub><mspace width="thickmathspace"></mspace><msup><mi>d</mi><mn>2</mn></msup><mrow><mo>(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo>)</mo></mrow></mrow></mtd><mtd columnalign="left"><mrow><mo>=</mo><msub><mi>max</mi><mi mathvariant="bold">y</mi></msub><mspace width="thickmathspace"></mspace><mi>cos</mi><mrow><mo>(</mo><mi>α</mi><mo>)</mo></mrow><mo>.</mo></mrow></mtd></mtr></mtable></math></script></div><div class="l">(13)</div></div><p></p><p id="P36">Such contrast normalization is consistent with neural models of primary visual cortex [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937479" role="button" aria-expanded="false" aria-haspopup="true">23</a>]. Cosine similarity measures were previously found to be effective for computational models of language [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937474" role="button" aria-expanded="false" aria-haspopup="true">24</a>] and face processing [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510583" role="button" aria-expanded="false" aria-haspopup="true">46</a>].</p><p id="P37"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F10/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Fig. 10</span></a> gives face recognition performance with both the ICA and the PCA-based representations. Recognition performance is also shown for the PCA-based representation using the first 20 PC vectors, which was the eigenface representation used by Pentland <em>et al</em>. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R51" rid="R51" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">51</a>]. Best performance for PCA was obtained using 200 coefficients. Excluding the first one, two, or three PCs did not improve PCA performance, nor did selecting intermediate ranges of components from 20 through 200. There was a trend for the ICA representation to give superior face recognition performance to the PCA representation with 200 components. The difference in performance was statistically significant for Test Set 3 (<em>Z</em> = 1.94, <em>p</em> = 0.05). The difference in performance between the ICA representation and the eigenface representation with 20 components was statistically significant over all three test sets (<em>Z</em> = 2.5, <em>p</em> &lt; 0.05) for Test Sets 1 and 2, and (<em>Z</em> =2.4 &lt; 0.05), <em>p</em> &lt; 0.05) for Test Set 3.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F10" co-legend-rid="lgnd_F10"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094234268752" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0013.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0013.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0013.jpg"></div></a><div id="largeobj_idm140094234268752" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F10/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Fig. 10</a></div><!--caption a4--><div><span>Percent correct face recognition for the ICA representation, Architecture I, using 200 ICs, the PCA representation using 200 PCs, and the PCA representation using 20 PCs. Groups are performances for Test Set 1, Test Set 2, and Test Set 3. Error bars are one standard deviation of the estimate of the success rate for a Bernoulli distribution.</span></div></div></div><p id="P38" class="p p-last">Recognition performance using different numbers of ICs was also examined by performing ICA on 20 to 200 image mixtures in steps of 20. Best performance was obtained by separating 200 ICs. In general, the more ICs were separated, the better the recognition performance. The basis images also became increasingly spatially local as the number of separated components increased.</p></div><div id="S10" class="sec sec-last"><h3>B. Subspace Selection</h3><p id="P39" class="p p-first">When all 200 components were retained, then PCA and ICA were working in the same subspace. However, as illustrated in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, when subsets of axes are selected, then ICA chooses a different subspace from PCA. The full benefit of ICA may not be tapped until ICA-defined subspaces are explored.</p><p id="P40">Face recognition performances for the PCA and ICA representations were next compared by selecting subsets of the 200 components by class discriminability.<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-17-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M23" overflow="scroll" role="math" style="width: 2.66em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.069em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.676em, 1002.07em, 2.611em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-399"><span class="mrow" id="MathJax-Span-400"><span class="mi" id="MathJax-Span-401" style="font-family: STIXGeneral;">Let</span><span class="mover" id="MathJax-Span-402" style="padding-left: 0.199em;"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.052em;"><span class="mi" id="MathJax-Span-403" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.377em; left: 0.052em;"><span class="mo" id="MathJax-Span-404" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.003em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-17"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M23" overflow="scroll"><mrow><mi>Let</mi><mover accent="true"><mi>x</mi><mo stretchy="false">‒</mo></mover></mrow></math></script> be the overall mean of a coefficient <em>b<sub>k</sub></em> across all faces, and <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-18-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M24" overflow="scroll" role="math" style="width: 1.036em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.199em, 1000.79em, 1.43em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-406"><span class="mrow" id="MathJax-Span-407"><span class="msub" id="MathJax-Span-408"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mover" id="MathJax-Span-409"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.052em;"><span class="mi" id="MathJax-Span-410" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.377em; left: 0.052em;"><span class="mo" id="MathJax-Span-411" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mi" id="MathJax-Span-412" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-18"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M24" overflow="scroll"><mrow><msub><mover accent="true"><mi>x</mi><mo stretchy="false">‒</mo></mover><mi>j</mi></msub></mrow></math></script> be the mean for person <em>j</em>. For both the PCA and ICA representations, we calculated the ratio of between-class to within-class variability <em>r</em> for each coefficient
</p><div class="disp-formula" id="FD14"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-19-Frame" tabindex="0" style=""><nobr><span class="math" id="M25" overflow="scroll" role="math" style="width: 6.252em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.924em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.233em, 1004.92em, 3.447em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-414"><span class="mrow" id="MathJax-Span-415"><span class="mi" id="MathJax-Span-416" style="font-family: STIXGeneral; font-style: italic;">r<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span class="mo" id="MathJax-Span-417" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mfrac" id="MathJax-Span-418" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 3.054em; height: 0px; margin-right: 0.101em; margin-left: 0.101em;"><span style="position: absolute; clip: rect(3.398em, 1002.96em, 4.284em, -999.998em); top: -4.673em; left: 50%; margin-left: -1.474em;"><span class="mrow" id="MathJax-Span-419"><span class="msub" id="MathJax-Span-420"><span style="display: inline-block; position: relative; width: 2.955em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-421" style="font-family: STIXGeneral; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mtext" id="MathJax-Span-422" style="font-size: 70.7%; font-family: STIXGeneral;">between</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1002.37em, 4.284em, -999.998em); top: -3.295em; left: 50%; margin-left: -1.179em;"><span class="mrow" id="MathJax-Span-423"><span class="msub" id="MathJax-Span-424"><span style="display: inline-block; position: relative; width: 2.365em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-425" style="font-family: STIXGeneral; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mtext" id="MathJax-Span-426" style="font-size: 70.7%; font-family: STIXGeneral;">within</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(0.888em, 1003.05em, 1.233em, -999.998em); top: -1.277em; left: 0.002em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.002em; border-top: 1.3px solid; width: 3.054em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.122em; border-left: 0px solid; width: 0px; height: 2.628em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-19"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M25" overflow="scroll"><mrow><mi>r</mi><mo>=</mo><mfrac><mrow><msub><mi>σ</mi><mtext>between</mtext></msub></mrow><mrow><msub><mi>σ</mi><mtext>within</mtext></msub></mrow></mfrac></mrow></math></script></div><div class="l">(14)</div></div><p>
where <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-20-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M26" overflow="scroll" role="math" style="width: 12.06em; display: inline-block;"><span style="display: inline-block; position: relative; width: 9.5em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.233em, 1009.5em, 3.054em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-428"><span class="mrow" id="MathJax-Span-429"><span class="msub" id="MathJax-Span-430"><span style="display: inline-block; position: relative; width: 2.955em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-431" style="font-family: STIXGeneral; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mtext" id="MathJax-Span-432" style="font-size: 70.7%; font-family: STIXGeneral;">between</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-433" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msub" id="MathJax-Span-434" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.103em, 1000.84em, 4.382em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-435" style="font-family: STIXGeneral; vertical-align: 0.002em;">∑</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.688em; left: 0.938em;"><span class="mi" id="MathJax-Span-436" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-437" style="padding-left: 0.199em;"><span class="msup" id="MathJax-Span-438"><span style="display: inline-block; position: relative; width: 3.89em; height: 0px;"><span style="position: absolute; clip: rect(2.955em, 1003.3em, 4.481em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-439"><span class="mo" id="MathJax-Span-440" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">(</span></span><span class="msub" id="MathJax-Span-441"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mover" id="MathJax-Span-442"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.052em;"><span class="mi" id="MathJax-Span-443" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.377em; left: 0.052em;"><span class="mo" id="MathJax-Span-444" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mi" id="MathJax-Span-445" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-446" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mover" id="MathJax-Span-447" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.052em;"><span class="mi" id="MathJax-Span-448" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.377em; left: 0.052em;"><span class="mo" id="MathJax-Span-449" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-450" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.574em; left: 3.447em;"><span class="mn" id="MathJax-Span-451" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.622em; border-left: 0px solid; width: 0px; height: 2.066em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-20"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M26" overflow="scroll"><mrow><msub><mi>σ</mi><mtext>between</mtext></msub><mo>=</mo><msub><mo>∑</mo><mi>j</mi></msub><mrow><msup><mrow><mo>(</mo><msub><mover accent="true"><mi>x</mi><mo stretchy="false">‒</mo></mover><mi>j</mi></msub><mo>−</mo><mover accent="true"><mi>x</mi><mo stretchy="false">‒</mo></mover><mo>)</mo></mrow><mn>2</mn></msup></mrow></mrow></math></script> is the variance of the <em>j</em> class means, and <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-21-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M27" overflow="scroll" role="math" style="width: 13.585em; display: inline-block;"><span style="display: inline-block; position: relative; width: 10.682em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.233em, 1010.68em, 3.054em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-453"><span class="mrow" id="MathJax-Span-454"><span class="msub" id="MathJax-Span-455"><span style="display: inline-block; position: relative; width: 2.365em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-456" style="font-family: STIXGeneral; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mtext" id="MathJax-Span-457" style="font-size: 70.7%; font-family: STIXGeneral;">within</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-458" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="msub" id="MathJax-Span-459" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.103em, 1000.84em, 4.382em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-460" style="font-family: STIXGeneral; vertical-align: 0.002em;">∑</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.688em; left: 0.938em;"><span class="mi" id="MathJax-Span-461" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msub" id="MathJax-Span-462" style="padding-left: 0.199em;"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.103em, 1000.84em, 4.382em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-463" style="font-family: STIXGeneral; vertical-align: 0.002em;">∑</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.688em; left: 0.938em;"><span class="mi" id="MathJax-Span-464" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-465" style="padding-left: 0.199em;"><span class="msup" id="MathJax-Span-466"><span style="display: inline-block; position: relative; width: 4.284em; height: 0px;"><span style="position: absolute; clip: rect(2.955em, 1003.69em, 4.481em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-467"><span class="mo" id="MathJax-Span-468" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">(</span></span><span class="msub" id="MathJax-Span-469"><span style="display: inline-block; position: relative; width: 0.938em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-470" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.445em;"><span class="mrow" id="MathJax-Span-471"><span class="mi" id="MathJax-Span-472" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span class="mi" id="MathJax-Span-473" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-474" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="msub" id="MathJax-Span-475" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.54em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mover" id="MathJax-Span-476"><span style="display: inline-block; position: relative; width: 0.544em; height: 0px;"><span style="position: absolute; clip: rect(3.398em, 1000.45em, 4.136em, -999.998em); top: -3.984em; left: 0.052em;"><span class="mi" id="MathJax-Span-477" style="font-family: STIXGeneral; font-style: italic;">x<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.377em; left: 0.052em;"><span class="mo" id="MathJax-Span-478" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.544em;"><span class="mi" id="MathJax-Span-479" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-480" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.574em; left: 3.841em;"><span class="mn" id="MathJax-Span-481" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.622em; border-left: 0px solid; width: 0px; height: 2.066em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-21"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M27" overflow="scroll"><mrow><msub><mi>σ</mi><mtext>within</mtext></msub><mo>=</mo><msub><mo>∑</mo><mi>j</mi></msub><msub><mo>∑</mo><mi>i</mi></msub><mrow><msup><mrow><mo>(</mo><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>−</mo><msub><mover accent="true"><mi>x</mi><mo stretchy="false">‒</mo></mover><mi>j</mi></msub><mo>)</mo></mrow><mn>2</mn></msup></mrow></mrow></math></script> is the sum of the variances within each class.</p><p id="P41">The class discriminability analysis was carried out using the 43 subjects for which four frontal view images were available. The ratios <em>r</em> were calculated separately for each test set, excluding the test images from the analysis. Both the PCA and ICA coefficients were then ordered by the magnitude of <em>r</em>. (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Fig. 11</span></a> top) compares the discriminability of the ICA coefficients to the PCA coefficients. The ICA coefficients consistently had greater class discriminability than the PCA coefficients.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F11" co-legend-rid="lgnd_F11"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094228142448" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0014.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0014.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0014.jpg"></a></div><div id="largeobj_idm140094228142448" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F11"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11">Fig. 11</a></div><!--caption a4--><div><span>Selection of components by class discriminability, Architecture II. Top: Discriminability of the ICA coefficients (solid lines) and discriminability of the PCA components (dotted lines) for the three test cases. Components were sorted by the magnitude of <em>r</em>. Bottom: Improvement in face recognition performance for the ICA and PCA representations using subsets of components selected by the class discriminability <em>r</em>. The improvement is indicated by the gray segments at the top of the bars.</span></div></div></div><p id="P42" class="p p-last">Face classification performance was compared using the <em>k</em> most discriminable components of each representation. (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F11/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Fig. 11</span></a> bottom) shows the best classification performance obtained for the PCA and ICA representations, which was with the 60 most discriminable components for the ICA representation, and the 140 most discriminable components for the PCA representation. Selecting subsets of coefficients by class discriminability improved the performance of the ICA representation, but had little effect on the performance of the PCA representation. The ICA representation again outperformed the PCA representation. The difference in recognition performance between the ICA and PCA representations was significant for Test Set 2 and Test Set 3, the two conditions that required recognition of images collected on a different day from the training set (<em>Z</em> = 2.9, <em>p</em> &lt; 0.05; <em>Z</em> = 3.4, <em>p</em> &lt; 0.01), respectively, when both subspaces were selected under the criterion of class discriminability. Here, the ICA-defined subspace encoded more information about facial identity than PCA-defined subspace.</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">V. Architecture II: A Factorial Face Code</h2><p id="P43" class="p p-first">The goal in Architecture I was to use ICA to find a set of spatially independent basis images. Although the basis images obtained in that architecture are approximately independent, the coefficients that code each face are not necessarily independent. Architecture II uses ICA to find a representation in which the coefficients used to code images are statistically independent, i.e., a factorial face code. Barlow and Atick have discussed advantages of factorial codes for encoding complex objects that are characterized by high-order combinations of features [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937475" role="button" aria-expanded="false" aria-haspopup="true">2</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>]. These include fact that the probability of any combination of features can be obtained from their marginal probabilities.</p><p id="P44">To achieve this goal, we organize the data matrix <em>X</em> so that rows represent different pixels and columns represent different images. [See (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> right)]. This corresponds to treating the columns of <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-22-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M28" overflow="scroll" role="math" style="width: 4.825em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.792em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.036em, 1003.79em, 2.808em, -999.998em); top: -2.36em; left: 0.002em;"><span class="mrow" id="MathJax-Span-483"><span class="mrow" id="MathJax-Span-484"><span class="mi" id="MathJax-Span-485" style="font-family: STIXGeneral; font-style: italic;">A</span><span class="mover" id="MathJax-Span-486" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 0.691em; height: 0px;"><span style="position: absolute; clip: rect(3.447em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-487" style="font-family: STIXGeneral;">=</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.25em, 1000.4em, 4.136em, -999.998em); top: -4.574em; left: 0.15em;"><span class="mi" id="MathJax-Span-488" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">Δ</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-489" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-490" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-491"><span class="mo" id="MathJax-Span-492" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-493" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-494" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.365em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 2.003em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-22"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M28" overflow="scroll"><mrow><mi>A</mi><mover><mo>=</mo><mi>Δ</mi></mover><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script> as a set of basis images. (See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F5/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). The ICA representations are in columns of <em>U</em> = <em>W<sub>I</sub>X</em>. Each column of <em>U</em> contains the coefficients of the basis images in <em>A</em> for reconstructing each image in <em>X</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F12/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F12" rid-ob="ob-F12" co-legend-rid="lgnd_F12"><span>Fig. 12</span></a>). ICA attempts to make the outputs, <em>U</em>, as independent as possible. Hence, <em>U</em> is a factorial code for the face images. The representational code for test images is obtained by
</p><div class="disp-formula" id="FD15"><div class="f"><em>W</em><sub><em>I</em></sub><em>X</em><sub>test</sub> =&nbsp;<em>U</em><sub>test</sub></div><div class="l">(15)</div></div><p>
where <em>X</em><sub>test</sub> is the zero-mean<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN7" rid="FN7" class=" fn">6</a></sup> matrix of test images, and <em>W<sub>I</sub></em> is the weight matrix found by performing ICA on the training images.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094230970960" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0008.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0008.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0008.jpg"></div></a><div id="largeobj_idm140094230970960" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a4--><div><span>Image synthesis model for Architecture II, based on [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510586" role="button" aria-expanded="false" aria-haspopup="true">43</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937469" role="button" aria-expanded="false" aria-haspopup="true">44</a>]. Each image in the dataset was considered to be a linear combination of underlying basis images in the matrix <em>A</em>. The basis images were each associated with a set of independent “causes,” given by a vector of coefficients in <em>S</em>. The basis images were estimated by <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-23-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M37" overflow="scroll" role="math" style="width: 5.033em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.94em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.479em, 1003.94em, 2.901em, -999.997em); top: -2.458em; left: 0.003em;"><span class="mrow" id="MathJax-Span-496"><span class="mrow" id="MathJax-Span-497"><span class="mi" id="MathJax-Span-498" style="font-family: STIXGeneral; font-weight: bold;">A</span><span class="mo" id="MathJax-Span-499" style="font-family: STIXGeneral; padding-left: 0.331em;">=</span><span class="msubsup" id="MathJax-Span-500" style="padding-left: 0.331em;"><span style="display: inline-block; position: relative; width: 1.917em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.99em, 4.158em, -999.997em); top: -3.989em; left: 0.003em;"><span class="mi" id="MathJax-Span-501" style="font-family: STIXGeneral; font-weight: bold;">W</span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span><span style="position: absolute; clip: rect(3.338em, 1000.93em, 4.158em, -999.997em); top: -4.317em; left: 0.987em;"><span class="mrow" id="MathJax-Span-502"><span class="mo" id="MathJax-Span-503" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-504" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span><span style="position: absolute; clip: rect(3.393em, 1000.33em, 4.158em, -999.997em); top: -3.716em; left: 0.987em;"><span class="mi" id="MathJax-Span-505" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.413em; border-left: 0px solid; width: 0px; height: 1.531em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-23"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M37" overflow="scroll"><mrow><mi mathvariant="bold">A</mi><mo>=</mo><msubsup><mi mathvariant="bold">W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script>, where <strong>W</strong><sub><em>I</em></sub> is the learned ICA weight matrix.</span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F12" co-legend-rid="lgnd_F12"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F12/" target="figure" rid-figpopup="F12" rid-ob="ob-F12"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094230821792" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0015.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0015.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0015.jpg"></div></a><div id="largeobj_idm140094230821792" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F12/" target="figure" rid-figpopup="F12" rid-ob="ob-F12"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F12/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F12"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F12/" target="figure" rid-figpopup="F12" rid-ob="ob-F12">Fig. 12</a></div><!--caption a4--><div><span>The factorial code representation consisted of the independent coefficients, <strong>u</strong>, for the linear combination of basis images in <em>A</em> that comprised each face image <strong>x</strong>.</span></div></div></div><p id="P45" class="p">In order to reduce the dimensionality of the input, instead of performing ICA directly on the 3000 image pixels, ICA was performed on the first 200 PCA coefficients of the face images. The first 200 PCs accounted for over 98% of the variance in the images. These coefficients <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-24-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M30" overflow="scroll" role="math" style="width: 2.217em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.725em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.052em, 1001.73em, 1.479em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-507"><span class="mrow" id="MathJax-Span-508"><span class="msubsup" id="MathJax-Span-509"><span style="display: inline-block; position: relative; width: 1.725em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-510" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-511" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1001.13em, 4.136em, -999.998em); top: -3.688em; left: 0.593em;"><span class="mn" id="MathJax-Span-512" style="font-size: 70.7%; font-family: STIXGeneral;">200</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.497em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-24"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M30" overflow="scroll"><mrow><msubsup><mi>R</mi><mn>200</mn><mi>T</mi></msubsup></mrow></math></script> comprised the columns of the input data matrix, where each coefficient had zero mean. The Architecture II representation for the training images was therefore contained in the columns of <em>U</em>, where
</p><div class="disp-formula" id="FD16"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-25-Frame" tabindex="0" style=""><nobr><span class="math" id="M31" overflow="scroll" role="math" style="width: 6.646em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.219em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.528em, 1005.17em, 2.955em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-514"><span class="mrow" id="MathJax-Span-515"><span class="msub" id="MathJax-Span-516"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-517" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.839em;"><span class="mi" id="MathJax-Span-518" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-519"><span style="display: inline-block; position: relative; width: 1.725em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-520" style="font-family: STIXGeneral; font-style: italic;">R</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.54em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-521" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1001.13em, 4.136em, -999.998em); top: -3.688em; left: 0.593em;"><span class="mn" id="MathJax-Span-522" style="font-size: 70.7%; font-family: STIXGeneral;">200</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-523" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mi" id="MathJax-Span-524" style="font-family: STIXGeneral; font-style: italic; padding-left: 0.298em;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span class="mo" id="MathJax-Span-525" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.497em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-25"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M31" overflow="scroll"><mrow><msub><mi>W</mi><mi>I</mi></msub><msubsup><mi>R</mi><mn>200</mn><mi>T</mi></msubsup><mo>=</mo><mi>U</mi><mo>.</mo></mrow></math></script></div><div class="l">(16)</div></div><p>
The ICA weight matrix <em>W<sub>I</sub></em> was 200 × 200, resulting in 200 coefficients in <em>U</em> for each face image, consisting of the outputs of each of the ICA filters.<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN8" rid="FN8" class=" fn">7</a></sup> The architecture II representation for test images was obtained in the columns of <em>U</em><sub>test</sub> as follows:
</p><div class="disp-formula" id="FD17"><div class="f"><em>W</em><sub><em>I</em></sub><em>R</em><sub>test</sub><sup><em>T</em></sup> =&nbsp;<em>U</em><sub>test</sub>.</div><div class="l">(17)</div></div><p>
The basis images for this representation consisted of the columns of <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-26-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M33" overflow="scroll" role="math" style="width: 4.825em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.792em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(1.036em, 1003.79em, 2.808em, -999.998em); top: -2.36em; left: 0.002em;"><span class="mrow" id="MathJax-Span-527"><span class="mrow" id="MathJax-Span-528"><span class="mi" id="MathJax-Span-529" style="font-family: STIXGeneral; font-style: italic;">A</span><span class="mover" id="MathJax-Span-530" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 0.691em; height: 0px;"><span style="position: absolute; clip: rect(3.447em, 1000.64em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-531" style="font-family: STIXGeneral;">=</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.25em, 1000.4em, 4.136em, -999.998em); top: -4.574em; left: 0.15em;"><span class="mi" id="MathJax-Span-532" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">Δ</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-533" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 1.873em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.89em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-534" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.94em, 4.136em, -999.998em); top: -4.328em; left: 0.987em;"><span class="mrow" id="MathJax-Span-535"><span class="mo" id="MathJax-Span-536" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-537" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.398em, 1000.35em, 4.136em, -999.998em); top: -3.688em; left: 0.839em;"><span class="mi" id="MathJax-Span-538" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.052em;"></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.365em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.434em; border-left: 0px solid; width: 0px; height: 2.003em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-26"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M33" overflow="scroll"><mrow><mi>A</mi><mover><mo>=</mo><mi>Δ</mi></mover><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script>. A sample of the basis images is shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F13/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F13" rid-ob="ob-F13" co-legend-rid="lgnd_F13"><span>Fig. 13</span></a>, where the PC reconstruction <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-27-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M34" overflow="scroll" role="math" style="width: 2.217em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.725em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.002em, 1001.73em, 1.479em, -999.998em); top: -0.982em; left: 0.002em;"><span class="mrow" id="MathJax-Span-540"><span class="mrow" id="MathJax-Span-541"><span class="msubsup" id="MathJax-Span-542"><span style="display: inline-block; position: relative; width: 1.725em; height: 0px;"><span style="position: absolute; clip: rect(3.201em, 1000.59em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-543" style="font-family: STIXGeneral; font-style: italic;">P</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1000.5em, 4.136em, -999.998em); top: -4.328em; left: 0.593em;"><span class="mi" id="MathJax-Span-544" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">A</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.349em, 1001.13em, 4.136em, -999.998em); top: -3.688em; left: 0.593em;"><span class="mn" id="MathJax-Span-545" style="font-size: 70.7%; font-family: STIXGeneral;">200</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.987em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.497em; border-left: 0px solid; width: 0px; height: 1.566em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-27"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M34" overflow="scroll"><mrow><msubsup><mi>P</mi><mn>200</mn><mi>A</mi></msubsup></mrow></math></script> was used to visualize them. In this approach, each column of the mixing matrix <em>W</em><sup>−1</sup> found by ICA attempts to get close to a cluster of images that look similar across pixels. Thus, this approach tends to generate basis images that look more face-like than the basis images generated by PCA, in that the bases found by ICA will average only images that look alike. Unlike the ICA output <em>U</em> the algorithm does not force the columns of <em>A</em> to be either sparse or independent. Indeed, the basis images in <em>A</em> have more global properties than the basis images in the ICA output of Architecture I shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F8/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Fig. 8</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F13" co-legend-rid="lgnd_F13"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F13/" target="figure" rid-figpopup="F13" rid-ob="ob-F13"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094227080656" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F13/" target="figure" rid-figpopup="F13" rid-ob="ob-F13"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0016.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0016.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0016.jpg"></a></div><div id="largeobj_idm140094227080656" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F13/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F13"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F13/" target="figure" rid-figpopup="F13" rid-ob="ob-F13">Fig. 13</a></div><!--caption a4--><div><span>Basis images for the ICA-factorial representation (columns of <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-28-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M38" overflow="scroll" role="math" style="width: 4.924em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.885em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.987em, 1003.88em, 2.791em, -999.997em); top: -2.349em; left: 0.003em;"><span class="mrow" id="MathJax-Span-547"><span class="mrow" id="MathJax-Span-548"><span class="mi" id="MathJax-Span-549" style="font-family: STIXGeneral; font-style: italic;">A</span><span class="mover" id="MathJax-Span-550" style="padding-left: 0.331em;"><span style="display: inline-block; position: relative; width: 0.714em; height: 0px;"><span style="position: absolute; clip: rect(3.448em, 1000.66em, 4.158em, -999.997em); top: -3.989em; left: 0.003em;"><span class="mo" id="MathJax-Span-551" style="font-family: STIXGeneral;">=</span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span><span style="position: absolute; clip: rect(3.284em, 1000.38em, 4.158em, -999.997em); top: -4.59em; left: 0.112em;"><span class="mi" id="MathJax-Span-552" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">Δ</span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-553" style="padding-left: 0.331em;"><span style="display: inline-block; position: relative; width: 1.917em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.93em, 4.158em, -999.997em); top: -3.989em; left: 0.003em;"><span class="mi" id="MathJax-Span-554" style="font-family: STIXGeneral; font-style: italic;">W<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span><span style="position: absolute; clip: rect(3.338em, 1000.93em, 4.158em, -999.997em); top: -4.317em; left: 0.987em;"><span class="mrow" id="MathJax-Span-555"><span class="mo" id="MathJax-Span-556" style="font-size: 70.7%; font-family: STIXGeneral;">−</span><span class="mn" id="MathJax-Span-557" style="font-size: 70.7%; font-family: STIXGeneral;">1</span></span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span><span style="position: absolute; clip: rect(3.393em, 1000.33em, 4.158em, -999.997em); top: -3.716em; left: 0.823em;"><span class="mi" id="MathJax-Span-558" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.994em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.354em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.413em; border-left: 0px solid; width: 0px; height: 2.017em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-28"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M38" overflow="scroll"><mrow><mi>A</mi><mover><mo>=</mo><mi>Δ</mi></mover><msubsup><mi>W</mi><mi>I</mi><mrow><mo>−</mo><mn>1</mn></mrow></msubsup></mrow></math></script>) obtained with Architecture II.</span></div></div></div><div id="S12" class="sec sec-last"><h3>A. Face Recognition Performance: Architecture II</h3><p id="P46" class="p p-first">Face recognition performance was again evaluated by the nearest neighbor procedure using cosines as the similarity measure. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F14/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F14" rid-ob="ob-F14" co-legend-rid="lgnd_F14"><span>Fig. 14</span></a> compares the face recognition performance using the ICA factorial code representation obtained with Architecture II to the independent basis representation obtained with Architecture I and to the PCA representation, each with 200 coefficients. Again, there was a trend for the ICA-factorial representation (ICA2) to outperform the PCA representation for recognizing faces across days. The difference in performance for Test Set 2 is significant (<em>Z</em> = 2.7, <em>p</em> &lt; 0.01). There was no significant difference in the performances of the two ICA representations.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F14" co-legend-rid="lgnd_F14"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F14/" target="figure" rid-figpopup="F14" rid-ob="ob-F14"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094229700000" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0017.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0017.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0017.jpg"></div></a><div id="largeobj_idm140094229700000" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F14/" target="figure" rid-figpopup="F14" rid-ob="ob-F14"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F14/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F14"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F14/" target="figure" rid-figpopup="F14" rid-ob="ob-F14">Fig. 14</a></div><!--caption a4--><div><span>Recognition performance of the factorial code ICA representation (ICA2) using all 200 coefficients, compared to the ICA independent basis representation (ICA1), and the PCA representation, also with 200 coefficients.</span></div></div></div><p id="P47">Class discriminability of the 200 ICA factorial coefficients was calculated according to (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD14" rid="FD14" class=" disp-formula">14</a>). Unlike the coefficients in the independent basis representation, the ICA-factorial coefficients did not differ substantially from each other according to discriminability <em>r</em>. Selection of subsets of components for the representation by class discriminability had little effect on the recognition performance using the ICA-factorial representation (see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F15/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F15" rid-ob="ob-F15" co-legend-rid="lgnd_F15"><span>Fig. 15</span></a>). The difference in performance between ICA1 and ICA2 for Test Set 3 following the discriminability analysis just misses significance (<em>Z</em> = 1.88, <em>p</em> = 0.06).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F15" co-legend-rid="lgnd_F15"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F15/" target="figure" rid-figpopup="F15" rid-ob="ob-F15"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094231032144" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0018.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0018.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0018.jpg"></div></a><div id="largeobj_idm140094231032144" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F15/" target="figure" rid-figpopup="F15" rid-ob="ob-F15"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F15/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F15"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F15/" target="figure" rid-figpopup="F15" rid-ob="ob-F15">Fig. 15</a></div><!--caption a4--><div><span>Improvement in recognition performance of the two ICA representations and the PCA representation by selecting subsets of components by class discriminability. Gray extensions show improvement over recognition performance using all 200 coefficients.</span></div></div></div><p id="P48" class="p p-last">In this implementation, we separated 200 components using 425 samples, which was a bare minimum. Test images were not used to learn the ICs, and thus our recognition results were not due to overlearning. Nevertheless, in order to determine whether the findings were an artifact due to small sample size, recognition performances were also tested after separating 85 rather than 200 components and, hence, estimating fewer weight parameters. The same overall pattern of results was obtained when 85 components were separated. Both ICA representations significantly outperformed the PCA representation on Test Sets 2 and 3. With 85 ICs, ICA1 obtained 87%, 62%, 58% correct performance, respectively, on Test Sets 1, 2, and 3, ICA2 obtained 85%, 76%, and 56% correct performance, whereas PCA obtained 85%, 56%, and 44% correct, respectively. Again, as found for 200 separated components, selection of subsets of components by class discriminability improved the performance of ICA1 to 86%, 78%, and 65%, respectively, and had little effect on the performances with the PCA and ICA2 representations. This suggests that the results were not simply an artifact due to small sample size.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">VI. Examination Of The ICA Representations</h2><div id="S14" class="sec sec-first"><h3>A. Mutual Information</h3><p id="P49" class="p p-first">A measure of the statistical dependencies of the face representations was obtained by calculating the mean mutual information between pairs of 50 basis images. Mutual information was calculated as
</p><div class="disp-formula" id="FD18"><div class="f"><em>I</em>(<strong>U</strong><sub>1</sub>,&nbsp;<strong>U</strong><sub>2</sub>) =&nbsp;<em>H</em>(<strong>U</strong><sub>1</sub>) +&nbsp;<em>H</em>(<strong>U</strong><sub>2</sub>)&nbsp;−&nbsp;<em>H</em>(<strong>U</strong><sub>1</sub>,&nbsp;<strong>U</strong><sub>2</sub>)</div><div class="l">(18)</div></div><p>
where <em>H</em>(<strong>U</strong><em><sub>i</sub></em>) = −<em>E</em>(log(<em>P<sub>Ui</sub></em>(<strong>U</strong><em><sub>i</sub></em>))).</p><p id="P50"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F16" rid-ob="ob-F16" co-legend-rid="lgnd_F16"><span>Fig. 16 (a)</span></a> compares the mutual information between <em>basis images</em> for the original gray-level images, the PC basis images, and the ICA basis images obtained in Architecture I. Principal component (PC) images are uncorrelated, but there are remaining high-order dependencies. The information maximization algorithm decreased these residual dependencies by more than 50%. The remaining dependence may be due to a mismatch between the logistic transfer function employed in the learning rule and the cumulative density function of the independent sources, the presence of sub-Gaussian sources, or the large number of free parameters to be estimated relative to the number of training images.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F16" co-legend-rid="lgnd_F16"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" rid-figpopup="F16" rid-ob="ob-F16"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094234829168" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" rid-figpopup="F16" rid-ob="ob-F16"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2898524_nihms-199383-f0019.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0019.jpg" title="Click on image to zoom" class="tileshop" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0019.jpg"></a></div><div id="largeobj_idm140094234829168" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F16"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" rid-figpopup="F16" rid-ob="ob-F16">Fig. 16</a></div><!--caption a4--><div><span>Pairwise mutual information. (a) Mean mutual information between basis images. Mutual information was measured between pairs of gray-level images, PC images, and independent basis images obtained by Architecture I. (b) Mean mutual information between coding variables. Mutual information was measured between pairs of image pixels in gray-level images, PCA coefficients, and ICA coefficients obtained by Architecture II.</span></div></div></div><p id="P51" class="p p-last"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F16" rid-ob="ob-F16" co-legend-rid="lgnd_F16"><span>Fig. 16 (b)</span></a> compares the mutual information between the <em>coding variables</em> in the ICA factorial representation obtained with Architecture II, the PCA representation, and gray-level images. For gray-level images, mutual information was calculated between pairs of pixel locations. For the PCA representation, mutual information was calculated between pairs of PC coefficients, and for the ICA factorial representation, mutual information was calculated between pairs of coefficients <em>u</em>. Again, there were considerable high-order dependencies remaining in the PCA representation that were reduced by more than 50% by the information maximization algorithm. The ICA representations obtained in these simulations are most accurately described not as “independent,” but as “redundancy reduced,” where the redundancy is less than half that in the PC representation.</p></div><div id="S15" class="sec sec-last"><h3>B. Sparseness</h3><p id="P52" class="p p-first">Field [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>] has argued that sparse distributed representations are advantageous for coding visual stimuli. Sparse representations are characterized by highly kurtotic response distributions, in which a large concentration of values are near zero, with rare occurrences of large positive or negative values in the tails. In such a code, the redundancy of the input is transformed into the redundancy of the response patterns of the the individual outputs. Maximizing sparseness without loss of information is equivalent to the minimum entropy codes discussed by Barlow [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>].<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN9" rid="FN9" class=" fn">8</a></sup></p><p id="P53">Given the relationship between sparse codes and minimum entropy, the advantages for sparse codes as outlined by Field [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>] mirror the arguments for independence presented by Barlow [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>]. Codes that minimize the number of active neurons can be useful in the detection of suspicious coincidences. Because a nonzero response of each unit is relatively rare, high-order relations become increasingly rare, and therefore, more informative when they are present in the stimulus. Field contrasts this with a compact code such as PCs, in which a few units have a relatively high probability of response, and therefore, high-order combinations among this group are relatively common. In a sparse distributed code, different objects are represented by which units are active, rather than by how much they are active. These representations have an added advantage in signal-to-noise, since one need only determine which units are active without regard to the precise level of activity. An additional advantage of sparse coding for face representations is storage in associative memory systems. Networks with sparse inputs can store more memories and provide more effective retrieval with partial information [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R10" rid="R10" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">10</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510568" role="button" aria-expanded="false" aria-haspopup="true">47</a>].</p><p id="P54" class="p p-last">The probability densities for the values of the coefficients of the two ICA representations and the PCA representation are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F17/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F17" rid-ob="ob-F17" co-legend-rid="lgnd_F17"><span>Fig. 17</span></a>. The sparseness of the face representations were examined by measuring the kurtosis of the distributions. Kurtosis is defined as the ratio of the fourth moment of the distribution to the square of the second moment, normalized to zero for the Gaussian distribution by subtracting 3
</p><div class="disp-formula" id="FD19"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-29-Frame" tabindex="0" style=""><nobr><span class="math" id="M36" overflow="scroll" role="math" style="width: 17.128em; display: inline-block;"><span style="display: inline-block; position: relative; width: 13.487em; height: 0px; font-size: 127%;"><span style="position: absolute; clip: rect(0.396em, 1013.44em, 4.628em, -999.998em); top: -2.458em; left: 0.002em;"><span class="mrow" id="MathJax-Span-560"><span class="mrow" id="MathJax-Span-561"><span class="mtext" id="MathJax-Span-562" style="font-family: STIXGeneral;">kurtosis</span><span class="mo" id="MathJax-Span-563" style="font-family: STIXGeneral; padding-left: 0.298em;">=</span><span class="mfrac" id="MathJax-Span-564" style="padding-left: 0.298em;"><span style="display: inline-block; position: relative; width: 6.941em; height: 0px; margin-right: 0.101em; margin-left: 0.101em;"><span style="position: absolute; clip: rect(2.758em, 1005.22em, 4.481em, -999.998em); top: -4.82em; left: 50%; margin-left: -2.606em;"><span class="mrow" id="MathJax-Span-565"><span class="munder" id="MathJax-Span-566"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.103em, 1000.84em, 4.382em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-567" style="font-family: STIXGeneral; vertical-align: 0.002em;">∑</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.688em; left: 0.938em;"><span class="mi" id="MathJax-Span-568" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-569" style="padding-left: 0.199em;"><span class="msup" id="MathJax-Span-570"><span style="display: inline-block; position: relative; width: 3.841em; height: 0px;"><span style="position: absolute; clip: rect(2.955em, 1003.25em, 4.481em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-571"><span class="mo" id="MathJax-Span-572" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">(</span></span><span class="msub" id="MathJax-Span-573"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.5em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-574" style="font-family: STIXGeneral; font-style: italic;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mi" id="MathJax-Span-575" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-576" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mover" id="MathJax-Span-577" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 0.495em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.5em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-578" style="font-family: STIXGeneral; font-style: italic;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.574em; left: 0.002em;"><span class="mo" id="MathJax-Span-579" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-580" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.574em; left: 3.398em;"><span class="mn" id="MathJax-Span-581" style="font-size: 70.7%; font-family: STIXGeneral;">4</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(2.463em, 1006.84em, 4.825em, -999.998em); top: -2.655em; left: 50%; margin-left: -3.393em;"><span class="mrow" id="MathJax-Span-582"><span class="msup" id="MathJax-Span-583"><span style="display: inline-block; position: relative; width: 6.843em; height: 0px;"><span style="position: absolute; clip: rect(2.66em, 1006.25em, 4.825em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-584"><span class="mo" id="MathJax-Span-585" style="vertical-align: -0.391em;"><span style="font-family: STIXSizeTwoSym;">(</span></span><span class="mrow" id="MathJax-Span-586"><span class="munder" id="MathJax-Span-587"><span style="display: inline-block; position: relative; width: 1.184em; height: 0px;"><span style="position: absolute; clip: rect(3.103em, 1000.84em, 4.382em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mo" id="MathJax-Span-588" style="font-family: STIXGeneral; vertical-align: 0.002em;">∑</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.688em; left: 0.938em;"><span class="mi" id="MathJax-Span-589" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mrow" id="MathJax-Span-590" style="padding-left: 0.199em;"><span class="msup" id="MathJax-Span-591"><span style="display: inline-block; position: relative; width: 3.841em; height: 0px;"><span style="position: absolute; clip: rect(2.955em, 1003.25em, 4.481em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mrow" id="MathJax-Span-592"><span class="mo" id="MathJax-Span-593" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">(</span></span><span class="msub" id="MathJax-Span-594"><span style="display: inline-block; position: relative; width: 0.79em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.5em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-595" style="font-family: STIXGeneral; font-style: italic;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -3.836em; left: 0.495em;"><span class="mi" id="MathJax-Span-596" style="font-size: 70.7%; font-family: STIXGeneral; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-597" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mover" id="MathJax-Span-598" style="padding-left: 0.249em;"><span style="display: inline-block; position: relative; width: 0.495em; height: 0px;"><span style="position: absolute; clip: rect(3.152em, 1000.5em, 4.136em, -999.998em); top: -3.984em; left: 0.002em;"><span class="mi" id="MathJax-Span-599" style="font-family: STIXGeneral; font-style: italic;">b</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(3.595em, 1000.5em, 3.939em, -999.998em); top: -4.574em; left: 0.002em;"><span class="mo" id="MathJax-Span-600" style="font-family: STIXGeneral;">‒</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span><span class="mo" id="MathJax-Span-601" style="vertical-align: -0.194em;"><span style="font-family: STIXSizeOneSym;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.574em; left: 3.398em;"><span class="mn" id="MathJax-Span-602" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span></span><span class="mo" id="MathJax-Span-603" style="vertical-align: -0.391em;"><span style="font-family: STIXSizeTwoSym;">)</span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; top: -4.87em; left: 6.4em;"><span class="mn" id="MathJax-Span-604" style="font-size: 70.7%; font-family: STIXGeneral;">2</span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.989em;"></span></span><span style="position: absolute; clip: rect(0.888em, 1006.94em, 1.233em, -999.998em); top: -1.277em; left: 0.002em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.002em; border-top: 1.3px solid; width: 6.941em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-605" style="font-family: STIXGeneral; padding-left: 0.249em;">−</span><span class="mn" id="MathJax-Span-606" style="font-family: STIXGeneral; padding-left: 0.249em;">3</span><span class="mo" id="MathJax-Span-607" style="font-family: STIXGeneral;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.463em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -2.622em; border-left: 0px solid; width: 0px; height: 5.128em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-29"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M36" overflow="scroll"><mrow><mtext>kurtosis</mtext><mo>=</mo><mfrac><mrow><munder><mo>∑</mo><mi>i</mi></munder><mrow><msup><mrow><mo>(</mo><msub><mi>b</mi><mi>i</mi></msub><mo>−</mo><mover accent="true"><mi>b</mi><mo stretchy="false">‒</mo></mover><mo>)</mo></mrow><mn>4</mn></msup></mrow></mrow><mrow><msup><mrow><mo stretchy="true">(</mo><mrow><munder><mo>∑</mo><mi>i</mi></munder><mrow><msup><mrow><mo>(</mo><msub><mi>b</mi><mi>i</mi></msub><mo>−</mo><mover accent="true"><mi>b</mi><mo stretchy="false">‒</mo></mover><mo>)</mo></mrow><mn>2</mn></msup></mrow></mrow><mo stretchy="true">)</mo></mrow><mn>2</mn></msup></mrow></mfrac><mo>−</mo><mn>3</mn><mo>.</mo></mrow></math></script></div><div class="l">(19)</div></div><p>
The kurtosis of the PCA representation was measured for the PC coefficients. The PCs of the face images had a kurtosis of 0.28. The coefficients, <em>b</em>, of the independent basis representation from Architecture I had a kurtosis of 1.25. Although the basis images in Architecture I had a sparse distribution of gray-level values, the face coefficients with respect to this basis were not sparse. In contrast, the coefficients <em>u</em> of the ICA factorial code representation from Architecture II were highly kurtotic at 102.9.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F17" co-legend-rid="lgnd_F17"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F17/" target="figure" rid-figpopup="F17" rid-ob="ob-F17"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094230720592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0020.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0020.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0020.jpg"></div></a><div id="largeobj_idm140094230720592" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F17/" target="figure" rid-figpopup="F17" rid-ob="ob-F17"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F17/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F17"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F17/" target="figure" rid-figpopup="F17" rid-ob="ob-F17">Fig. 17</a></div><!--caption a4--><div><span>Kurtosis (sparseness) of ICA and PCA representations.</span></div></div></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">VII. Combined ICA Recognition System</h2><p id="P55" class="p p-first">Given that the two ICA representations gave similar recognition performances, we examined whether the two representations gave similar patterns of errors on the face images. There was a significant tendency for the two algorithms to misclassify the same images. The probability that the ICA-factorial representation (ICA2) made an error given that the ICA1 representation made an error was 0.72, 0.88, and 0.89, respectively, for the three test sets. These conditional error rates were significantly higher than the marginal error rates (<em>Z</em> = 7.4, <em>p</em> &lt; 0.001; <em>Z</em> = 3.4, <em>p</em> &lt; 0.001; <em>Z</em> = 2.8, <em>p</em> &lt; 0.01), respectively. Examples of successes and failures of the two algorithms are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F18/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F18" rid-ob="ob-F18" co-legend-rid="lgnd_F18"><span>Fig. 18</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F18" co-legend-rid="lgnd_F18"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F18/" target="figure" rid-figpopup="F18" rid-ob="ob-F18"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094231065024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0021.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0021.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0021.jpg"></div></a><div id="largeobj_idm140094231065024" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F18/" target="figure" rid-figpopup="F18" rid-ob="ob-F18"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F18/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F18"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F18/" target="figure" rid-figpopup="F18" rid-ob="ob-F18">Fig. 18</a></div><!--caption a4--><div><span>Recognition successes and failures. {left) Two face image pairs which both ICA algorithms correctly recognized. (right) Two face image pairs that were misidentified by both ICA algorithms. Images from the FERET face database were reprinted with permission from J. Phillips.</span></div></div></div><p id="P56">When the two algorithms made errors, however, they did not assign the same incorrect identity. Out of a total of 62 common errors between the two systems, only once did both algorithms assign the same incorrect identity. The two representations can, therefore, used in conjunction to provide a reliability measure, where classifications are accepted only if both algorithms gave the same answer. The ICA recognition system using this reliability criterion gave a performance of 100%, 100%, and 97% for the three test sets, respectively, which is an overall classification performance of 99.8%. 400 out of the total of 500 test images met criterion.</p><p id="P57" class="p p-last">Because the confusions made by the two algorithms differed, a combined classifier was employed in which the similarity between a test image and a gallery image was defined as <em>c</em><sub>1</sub> + <em>c</em><sub>2</sub>, where <em>c</em><sub>1</sub> and <em>c</em><sub>2</sub> correspond to the similarity measure in <em>c</em> in (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD12" rid="FD12" class=" disp-formula">12</a>) for ICA1 and ICA2, respectively. Class discriminability analysis was carried out on ICA1 and ICA2 before calculating <em>c</em><sub>1</sub> and <em>c</em><sub>2</sub>. Performance of the combined classifier is shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F19/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F19" rid-ob="ob-F19" co-legend-rid="lgnd_F19"><span>Fig. 19</span></a>. The combined classifier improved performance to 91.0%, 88.9%, and 81.0% for the three test cases, respectively. The difference in performance between the combined ICA classifier and PCA was significant for all three test sets (<em>Z</em> = 2.7, <em>p</em> &lt; 0.01; <em>Z</em> = 3.7, <em>p</em> &lt; 0.001; <em>Z</em> = 3.7; <em>p</em> &lt; 0.001).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F19" co-legend-rid="lgnd_F19"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F19/" target="figure" rid-figpopup="F19" rid-ob="ob-F19"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094233467488" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0022.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0022.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0022.jpg"></div></a><div id="largeobj_idm140094233467488" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F19/" target="figure" rid-figpopup="F19" rid-ob="ob-F19"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F19/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F19"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F19/" target="figure" rid-figpopup="F19" rid-ob="ob-F19">Fig. 19</a></div><!--caption a4--><div><span>Face recognition performance of the ocmbined ICA classifier, compared to the individual classifiers for ICA1, ICA2, and PCA.</span></div></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">VIII. Discussion</h2><p id="P58" class="p p-first">Much of the information that perceptually distinguishes faces is contained in the higher order statistics of the images, i.e., the phase spectrum. The basis images developed by PCA depend only on second-order images statistics and, thus, it is desirable to find generalizations of PCA that are sensitive to higher order image statistics. In this paper, we explored one such generalization: Bell and Sejnowski's ICA algorithm. We explored two different architectures for developing image representations of faces using ICA. Architecture I treated images as random variables and pixels as random trials. This architecture was related to the one used by Bell and Sejnowski to separate mixtures of auditory signals into independent sound sources. Under this architecture, ICA found a basis set of statistically independent images. The images in this basis set were sparse and localized in space, resembling facial features. Architecture II treated pixels as random variables and images as random trials. Under this architecture, the image coefficients were approximately independent, resulting in a factorial face code.</p><p id="P59">Both ICA representations outperformed the “eigenface” representation [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_367027316" role="button" aria-expanded="false" aria-haspopup="true">57</a>], which was based on PC analysis, for recognizing images of faces sampled on a different day from the training images. A classifier that combined the two ICA representations outperformed eigenfaces on all test sets. Since ICA allows the basis images to be nonorthogonal, the angles and distances between images differ between ICA and PCA. Moreover, when subsets of axes are selected, ICA defines a different subspace than PCA. We found that when selecting axes according to the criterion of class discriminability, ICA-defined subspaces encoded more information about facial identity than PCA-defined subspaces.</p><p id="P60">ICA representations are designed to maximize information transmission in the presence of noise and, thus, they may be more robust to variations such as lighting conditions, changes in hair, make-up, and facial expression, which can be considered forms of noise with respect to the main source of information in our face database: the person's identity. The robust recognition across different days is particularly encouraging, since most applications of automated face recognition contain the noise inherent to identifying images collected on a different day from the sample images.</p><p id="P61">The purpose of the comparison in this paper was to examine ICA and PCA-based representations under identical conditions. A number of methods have been presented for enhancing recognition performance with eigenfaces (e.g., [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R41" rid="R41" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">41</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R51" rid="R51" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">51</a>]). ICA representations can be used in place of eigenfaces in these techniques. It is an open question as to whether these techniques would enhance performance with PCA and ICA equally, or whether there would be interactions between the type of enhancement and the representation.</p><p id="P62">A number of research groups have independently tested the ICA representations presented here and in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R9" rid="R9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a>]. Liu and Wechsler [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>], and Yuen and Lai [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R61" rid="R61" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">61</a>] both supported our findings that ICA outperformed PCA. Moghaddam [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R41" rid="R41" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">41</a>] employed Euclidean distance as the similarity measure instead of cosines. Consistent with our findings, there was no significant difference between PCA and ICA using Euclidean distance as the similarity measure. Cosines were not tested in that paper. A thorough comparison of ICA and PCA using a large set of similarity measures was recently conducted in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R17" rid="R17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17</a>], and supported the advantage of ICA for face recognition.</p><p id="P63">In Section V, ICA provided a set of statistically independent coefficients for coding the images. It has been argued that such a factorial code is advantageous for encoding complex objects that are characterized by high-order combinations of features, since the prior probability of any combination of features can be obtained from their individual probabilities [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937478" role="button" aria-expanded="false" aria-haspopup="true">2</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>]. According to the arguments of both Field [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>] and Barlow [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>], the ICA-factorial representation (Architecture II) is a more optimal object representation than the Architecture I representation given its sparse, factorial properties. Due to the difference in architecture, the ICA-factorial representation always had fewer training samples to estimate the same number of free parameters as the Architecture I representation. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F16/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F16" rid-ob="ob-F16" co-legend-rid="lgnd_F16"><span>Fig. 16</span></a> shows that the residual dependencies in the ICA-factorial representation were higher than in the Architecture I representation. The ICA-factorial representation may prove to have a greater advantage given a much larger training set of images. Indeed, this prediction has born out in recent experiments with a larger set of FERET face images [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R17" rid="R17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17</a>]. It also is possible that the factorial code representation may prove advantageous with more powerful recognition engines than nearest neighbor on cosines, such as a Bayesian classifier. An image set containing many more frontal view images of each subject collected on different days will be needed to test that hypothesis.</p><p id="P64">In this paper, the number of sources was controlled by reducing the dimensionality of the data through PCA prior to performing ICA. There are two limitations to this approach [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R55" rid="R55" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">55</a>]. The first is the reverse dimensionality problem. It may not be possible to linearly separate the independent sources in smaller subspaces. Since we retained 200 dimensions, this may not have been a serious limitation of this implementation. Second, it may not be desirable to throw away subspaces of the data with low power such as the higher PCs. Although low in power, these subspaces may contain ICs, and the property of the data we seek is independence, not amplitude. Techniques have been proposed for separating sources on projection planes without discarding any ICs of the data [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R55" rid="R55" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">55</a>]. Techniques for estimating the number of ICs in a dataset have also recently been proposed [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R26" rid="R26" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">26</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R40" rid="R40" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">40</a>].</p><p id="P65">The information maximization algorithm employed to perform ICA in this paper assumed that the underlying “causes” of the pixel gray-levels in face images had a super-Gaussian (peaky) response distribution. Many natural signals, such as sound sources, have been shown to have a super-Gaussian distribution [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510585" role="button" aria-expanded="false" aria-haspopup="true">11</a>]. We employed a logistic source model which has shown in practice to be sufficient to separate natural signals with super-Gaussian distributions [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510581" role="button" aria-expanded="false" aria-haspopup="true">11</a>]. The underlying “causes” of the pixel gray-levels in the face images are unknown, and it is possible that better results could have been obtained with other source models. In particular, any sub-Gaussian sources would have remained mixed. Methods for separating sub-Gaussian sources through information maximization have been developed [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510579" role="button" aria-expanded="false" aria-haspopup="true">30</a>]. A future direction of this research is to examine sub-Gaussian components of face images.</p><p id="P66">The information maximization algorithm employed in this work also assumed that the pixel values in face images were generated from a linear mixing process. This linear approximation has been shown to hold true for the effect of lighting on face images [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R21" rid="R21" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">21</a>]. Other influences, such as changes in pose and expression may be linearly approximated only to a limited extent. Nonlinear ICA in the absence of prior constraints is an ill-conditioned problem, but some progress has been made by assuming a linear mixing process followed by parametric nonlinear functions [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R31" rid="R31" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">31</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R59" rid="R59" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">59</a>]. An algorithm for nonlinear ICA based on kernel methods has also recently been presented [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R4" rid="R4" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">4</a>]. Kernel methods have already shown to improve face recognition performance with PCA and Fisherfaces [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R60" rid="R60" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">60</a>]. Another future direction of this research is to examine nonlinear ICA representations of faces.</p><p id="P67">Unlike PCA, the ICA using Architecture I found a spatially local face representation. Local feature analysis (LFA) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R50" rid="R50" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">50</a>] also finds local basis images for faces, but using second-order statistics. The LFA basis images are found by performing whitening (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FD4" rid="FD4" class=" disp-formula">4</a>) on the PC axes, followed by a rotation to topographic correspondence with pixel location. The LFA kernels are not sensitive to the high-order dependencies in the face image ensemble, and in tests to date, recognition performance with LFA kernels has not significantly improved upon PCA [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937471" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. Interestingly, down-sampling methods based on sequential information maximization significantly improve performance with LFA [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R49" rid="R49" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">49</a>].</p><p id="P68">ICA outputs using Architecture I were sparse in space (within image across pixels) while the ICA outputs using Architecture II were sparse across images. Hence Architecture I produced local basis images, but the face codes were not sparse, while Architecture II produced sparse face codes, but with holistic basis images. A representation that has recently appeared in the literature, nonnegative matrix factorization (NMF) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510571" role="button" aria-expanded="false" aria-haspopup="true">28</a>], produced local basis images and sparse face codes.<sup><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#FN10" rid="FN10" class=" fn">9</a></sup> While this representation is interesting from a theoretical perspective, it has not yet proven useful for recognition. Another innovative face representation employs products of experts in restricted Boltzmann machines (RBMs). This representation also finds local features when nonnegative weight constraints are employed [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R56" rid="R56" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">56</a>]. In experiments to date, RBMs outperformed PCA for recognizing faces across changes in expression or addition/removal of glasses, but performed more poorly for recognizing faces across different days. It is an open question as to whether sparseness and local features are desirable objectives for face recognition in and of themselves. Here, these properties emerged from an objective of independence.</p><p id="P69">Capturing more likelihood may be a good principle for generating unsupervised representations which can be later used for classification. As mentioned in Section II, PCA and ICA can be derived as generative models of the data, where PCA uses Gaussian sources, and ICA typically uses sparse sources. It has been shown that for many natural signals, ICA is a better model in that it assigns higher likelihood to the data than PCA [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R32" rid="R32" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">32</a>]. The ICA basis dimensions presented here may have captured more likelihood of the face images than PCA, which provides a possible explanation for the superior performance of ICA for face recognition in this study.</p><p id="P70">The ICA representations have a degree of biological relevance. The information maximization learning algorithm was developed from the principle of optimal information transfer in neurons with sigmoidal transfer functions. It contains a Hebbian correlational term between the nonlinearly transformed outputs and weighted feedback from the linear outputs [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510584" role="button" aria-expanded="false" aria-haspopup="true">12</a>]. The biological plausibility of the learning algorithm, however, is limited by fact that the learning rule is nonlocal. Local learning rules for ICA are presently under development [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R34" rid="R34" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">34</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R38" rid="R38" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">38</a>].</p><p id="P71">The principle of independence, if not the specific learning algorithm employed here [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510576" role="button" aria-expanded="false" aria-haspopup="true">12</a>], may have relevance to face and object representations in the brain. Barlow [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>] and Atick [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937477" role="button" aria-expanded="false" aria-haspopup="true">2</a>] have argued for redundancy reduction as a general coding strategy in the brain. This notion is supported by the findings of Bell and Sejnowski [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510575" role="button" aria-expanded="false" aria-haspopup="true">12</a>] that image bases that produce independent outputs from natural scenes are local oriented spatially opponent filters similar to the response properties of V1 simple cells. Olshausen and Field [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510590" role="button" aria-expanded="false" aria-haspopup="true">43</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937473" role="button" aria-expanded="false" aria-haspopup="true">44</a>] obtained a similar result with a sparseness objective, where there is a close information theoretic relationship between sparseness and independence [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5" rid="R5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510589" role="button" aria-expanded="false" aria-haspopup="true">12</a>]. Conversely, it has also been shown that Gabor filters, which closely model the responses of V1 simple cells, separate high-order dependencies [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510573" role="button" aria-expanded="false" aria-haspopup="true">18</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R54" rid="R54" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">54</a>]. (See [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R6" rid="R6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a>] for a more detailed discussion). In support of the relationship between Gabor filters and ICA, the Gabor and ICA Architecture I representations significantly outperformed more than eight other image representations on a task of facial expression recognition, and performed equally well to each other [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R8" rid="R8" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937470" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. There is also psychophysical support for the relevance of independence to face representations in the brain. The ICA Architecture I representation gave better correspondence with human perception of facial similarity than both PCA and nonnegative matrix factorization [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R22" rid="R22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>].</p><p id="P72" class="p p-last">Desirable filters may be those that are adapted to the patterns of interest and capture interesting structure [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510566" role="button" aria-expanded="false" aria-haspopup="true">33</a>]. The more the dependencies that are encoded, the more structure that is learned. Information theory provides a means for capturing interesting structure. Information maximization leads to an efficient code of the environment, resulting in more learned structure. Such mechanisms predict neural codes in both vision [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510562" role="button" aria-expanded="false" aria-haspopup="true">12</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301510587" role="button" aria-expanded="false" aria-haspopup="true">43</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329937476" role="button" aria-expanded="false" aria-haspopup="true">58</a>] and audition [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R32" rid="R32" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">32</a>]. The research presented here found that face representations in which high-order dependencies are separated into individual coefficients gave superior recognition performance to representations which only separate second-order redundancies.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094228048304" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0007.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-f0007.jpg" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-f0007.jpg"></div></a><div id="largeobj_idm140094228048304" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a4--><div><span>Image synthesis model for Architecture I. To find a set of IC images, the images in <em>X</em> are considered to be a linear combination of statistically independent basis images, <em>S</em>, where <em>A</em> is an unknown mixing matrix. The basis images were estimated as the learned ICA output <em>U</em>.</span></div></div></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Acknowledgment</h2><div class="sec"><p id="__p1">The authors are grateful to M. Lades, M. McKeown, M. Gray, and T.-W. Lee for helpful discussions on this topic, and valuable comments on earlier drafts of this paper.</p><p id="__p2">This work was supported by University of California Digital Media Innovation Program D00-10084, the National Science Foundation under Grants 0086107 and IIT-0223052, the National Research Service Award MH-12417-02, the Lawrence Livermore National Laboratories ISCR agreement B291528, and the Howard Hughes Medical Institute.</p></div></div><div id="d35e3003" class="tsec sec headless whole_rhythm"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="d35e3003title">Biographies</h2><div class="bio" id="d35e3003"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094238513120" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0001.gif" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0001.gif" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-b0001.gif"></div><div id="largeobj_idm140094238513120" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/">Open in a separate window</a></div><p></p><p id="__p4"><strong>Marian Stewart Bartlett</strong> (M'99) received the B.S. degree in mathematics and computer science from Middlebury College, Middlebury, VT, in 1988 and the Ph.D. degree in cognitive science and psychology from the University of California-San Diego, La Jolla, in 1998. Her dissertation work was conducted with T. Sejnowski at the Salk Institute.</p><p id="__p5" class="p p-last">She is an Assistant Research Professor at the Institute for Neural Computation, University of California-San Diego. Her interests include approaches to image analysis through unsupervised learning, with a focus on face recognition and expression analysis. She is presently exploring probabilistic dynamical models and their application to facial expression analysis at the University of California-San Diego. She has also studied perceptual and cognitive processes with V.S. Ramachandran at the University of California-San Diego, the Cognitive Neuroscience Section of the National Institutes of Health, the Department of Brain and Cognitive Sciences at Massachusetts Institute of Technology, Cambridge, and the Brain and Perception Laboratory at the University of Bristol, U.K.</p></div><div class="bio" id="d35e3012"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094231063968" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0002.gif" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0002.gif" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-b0002.gif"></div><div id="largeobj_idm140094231063968" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/">Open in a separate window</a></div><p></p><p id="__p7"><strong>Javier R. Movellan</strong> (M'99) was born in Palencia, Spain, and received the B.S. degree from the Universidad Autonoma de Madrid, Madrid, Spain. He was a Fulbright Scholar at the University of California-Berkeley, Berkeley, and received the Ph.D. degree from the same university in 1989.</p><p id="__p8" class="p p-last">He was a Research Associate with Carnegie-Mellon University, Pittsburgh, PA, from 1989 to 1993, and an Assistant Professor with the Department of Cognitive Science, University of California-San Diego (USCD), La Jolla, from 1993 to 2001. He currently is a Research Associate with the Institute for Neural Computation and head of the Machine Perception Laboratory at UCSD. His research interests include the development of perceptual computer interfaces (i.e., system that recognize and react to natural speech commands, expressions, gestures, and body motions), analyzing the statistical structure of natural signals in order to help understand how the brain works, and the application of stochastic processes and probability theory to the study of the brain, behavior, and computation.</p></div><div class="bio" id="d35e3021"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140094224950048" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0003.gif" title="An external file that holds a picture, illustration, etc.
Object name is nihms-199383-b0003.gif" src="./Face Recognition by Independent Component Analysis_files/nihms-199383-b0003.gif"></div><div id="largeobj_idm140094224950048" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/">Open in a separate window</a></div><p></p><p id="__p10"><strong>Terrence J. Sejnowski</strong> (S'83–SM'91–F'00) received the B.S. degree in physics from the Case Western Reserve University, Cleveland, OH, and the Ph.D. degree in physics from Princeton University, Princeton, NJ, in 1978.</p><p id="__p11">In 1982, he joined the faculty of the Department of Biophysics at Johns Hopkins University, Baltimore, MD. He is an Investigator with the Howard Hughes Medical Institute and a Professor at The Salk Institute for Biological Studies, La Jolla, CA, where he directs the Computational Neurobiology Laboratory, and Professor of Biology at the University of California-San Diego, La Jolla. The long-range goal his research is to build linking principles from brain to behavior using computational models. This goal is being pursued with a combination of theoretical and experimental approaches at several levels of investigation ranging from the biophysical level to the systems level. The issues addressed by this research include how sensory information is represented in the visual cortex.</p><p id="__p12" class="p p-last">Dr. Sejnowski received the IEEE Neural Networks Pioneer Award in 2002.</p></div> </div><div id="idm140094242859488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140094242859488title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"><sup>1</sup>Preliminary versions of this work appear in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R7" rid="R7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">7</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R9" rid="R9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a>]. A longer discussion of unsupervised learning for face recognition appears in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R6" rid="R6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a>].</p><p class="fn sec" id="FN3"><sup>2</sup>We use the principal square root, which is the unique square root for which every eigenvalue has nonnegative real part.</p><p class="fn sec" id="FN4"><sup>3</sup><em>B</em><sub>test</sub> could potentially be obtained without calculating a pseudoinverse by normalizing the length of the rows of <em>U</em>, thereby making <em>U</em> approximately orthonormal, and calculating <em>B</em><sub>test</sub> = <em>X</em><sub>test </sub><em>U<sup>T</sup></em>. However, if ICA did not remove all of the second-order dependencies then <em>U</em> will not be precisely orthonormal.</p><p class="fn sec" id="FN5"><sup>4</sup>In pilot work, we found that face recognition performance improved with the number of components separated. We chose 200 components as the largest number to separate within our processing limitations.</p><p class="fn sec" id="FN6"><sup>5</sup>Although PCA already removed the covariances in the data, the variances were not equalized. We, therefore, retained the sphering step.</p><p class="fn sec" id="FN7"><sup>6</sup>Here, each pixel has zero mean.</p><p class="fn sec" id="FN8"><sup>7</sup>An image filter is <em>f</em>(x) is defined as <em>f</em>(x) = w · x.</p><p class="fn sec" id="FN9"><sup>8</sup>Information maximization is consistent with minimum entropy coding. By maximizing the <em>joint</em> entropy of the output, the entropies of the <em>individual</em> outputs tend to be minimized.</p><p class="fn sec" id="FN10"><sup>9</sup>Although the NMF codes were sparse, they were not a minimum entropy code (an independent code) as the objective function did not maximize sparseness while preserving information.</p></div></div><div id="idm140094221320112" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140094221320112title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">[1] <span class="element-citation">Amari S, Cichocki A, Yang HH.  <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 8. MIT Press; Cambridge, MA: 1996. A new learning algorithm for blind signal separation.</span></div><div class="ref-cit-blk half_rhythm" id="R2">[2] <span class="element-citation">Atick JJ. Could information theory provide an ecological theory of sensory processing? <span><span class="ref-journal">Network. </span>1992;<span class="ref-vol">3</span>:213–251.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/22149669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">[3] <span class="element-citation">Atick JJ, Redlich AN. What does the retina know about natural scenes? <span><span class="ref-journal">Neural Comput. </span>1992;<span class="ref-vol">4</span>:196–210.</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">[4] <span class="element-citation">Bach FR, Jordan MI. Kernel independent component analysis. <span><span class="ref-journal">J. Machine Learning Res. </span>2002;<span class="ref-vol">3</span>:1–48.</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">[5] <span class="element-citation">Barlow HB. Unsupervised learning. <span><span class="ref-journal">Neural Comput. </span>1989;<span class="ref-vol">1</span>:295–311.</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">[6] <span class="element-citation">Bartlett MS.  <span class="ref-journal">Face Image Analysis by Unsupervised Learning.</span> vol. 612. Kluwer; Boston, MA: 2001. Kluwer International Series on Engineering and Computer Science.</span></div><div class="ref-cit-blk half_rhythm" id="R7">[7] <span class="element-citation">Bartlett MS.  <span class="ref-journal">Ph.D. dissertation.</span> Univ. California-San Diego; La Jolla: 1998. Face Image Analysis by Unsupervised Learning and Redundancy Reduction.</span></div><div class="ref-cit-blk half_rhythm" id="R8">[8] <span class="element-citation">Bartlett MS, Donato GL, Movellan JR, Hager JC, Ekman P, Sejnowski TJ.  Image representations for facial expression coding. In: Solla SA, Leen TK, Muller K-R, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 12. MIT Press; Cambridge, MA: 2000. </span></div><div class="ref-cit-blk half_rhythm" id="R9">[9] <span class="element-citation">Bartlett MS, Lades HM, Sejnowski TJ. Independent component representations for face recognition. In: Rogowitz T, Pappas B, editors. Proc. SPIE Symp. Electon. Imaging: Science Technology—Human Vision and Electronic Imaging III; San Jose, CA. 1998. pp. 528–539.</span></div><div class="ref-cit-blk half_rhythm" id="R10">[10] <span class="element-citation">Baum EB, Moody J, Wilczek F. Internal representaions for associative memory. <span><span class="ref-journal">Biol. Cybern. </span>1988;<span class="ref-vol">59</span>:217–228.</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">[11] <span class="element-citation">Bell AJ, Sejnowski TJ. An information-maximization approach to blind separation and blind deconvolution. <span><span class="ref-journal">Neural Comput. </span>1995;<span class="ref-vol">7</span>(6):1129–1159.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">[12] <span class="element-citation">Bell AJ, Sejnowski TJ. The independent components of natural scenes are edge filters. <span><span class="ref-journal">Vision Res. </span>1997;<span class="ref-vol">37</span>(23):3327–3338.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2882863/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">[13] <span class="element-citation">Cichocki A, Unbehauen R, Rummert E. Robust learning algorithm for blind separation of signals. <span><span class="ref-journal">Electron. Lett. </span>1994;<span class="ref-vol">30</span>(7):1386–1387.</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">[14] <span class="element-citation">Comon P. Independent component analysis—A new concept? <span><span class="ref-journal">Signal Processing. </span>1994;<span class="ref-vol">36</span>:287–314.</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">[15] <span class="element-citation">Cottrell G, Metcalfe J.  Face, gender and emotion recognition using holons. In: Touretzky D, editor. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 3. Morgan Kaufmann; San Mateo, CA: 1991. pp. 564–571.</span></div><div class="ref-cit-blk half_rhythm" id="R16">[16] <span class="element-citation">Donato G, Bartlett M, Hager J, Ekman P, Sejnowski T. Classifying facial actions. <span><span class="ref-journal">IEEE Trans. Pattern Anal. Machine Intell. </span>1999 Oct.<span class="ref-vol">21</span>:974–989.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3008166/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/21188284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">[17] <span class="element-citation">Draper BA, Baek K, Bartlett MS, Beveridge JR.  <span class="ref-journal">Comput. Vision Image Understanding (Special Issue on Face Recognition)</span> 2002. Recognizing faces with PCA and ICA. submitted for publication.</span></div><div class="ref-cit-blk half_rhythm" id="R18">[18] <span class="element-citation">Field DJ. Relations between the statistics of natural images and the response properties of cortical cells. <span><span class="ref-journal">J. Opt. Soc. Amer. A. </span>1987;<span class="ref-vol">4</span>:2379–94.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/3430225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">[19] <span class="element-citation">Field DJ. What is the goal of sensory coding? <span><span class="ref-journal">Neural Comput. </span>1994;<span class="ref-vol">6</span>:559–601.</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">[20] <span class="element-citation">Girolami M.  <span class="ref-journal">Advances in Independent Component Analysis.</span> Springer-Verlag; Berlin, Germany: 2000. </span></div><div class="ref-cit-blk half_rhythm" id="R21">[21] <span class="element-citation">Hallinan P.  <span class="ref-journal">Ph.D. dissertation.</span> Harvard Univ.; Cambridge, MA: 1995. A Deformable Model for Face Recognition Under Arbitrary Lighting Conditions.</span></div><div class="ref-cit-blk half_rhythm" id="R22">[22] <span class="element-citation">Hancock P.  <span class="ref-journal">British Psych. Soc., Cognitive Section.</span> Univ. Essex; Essex, U.K.: 2000. Alternative representations for faces.</span></div><div class="ref-cit-blk half_rhythm" id="R23">[23] <span class="element-citation">Heeger DJ. Normalization of cell responses in cat striate cortex. <span><span class="ref-journal">Visual Neurosci. </span>1992;<span class="ref-vol">9</span>:181–197.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/1504027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">[24] <span class="element-citation">Hinton G, Shallice T. Lesioning an attractor network: Investigations of acquired dyslexia. <span><span class="ref-journal">Psych. Rev. </span>1991;<span class="ref-vol">98</span>(1):74–95.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/2006233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">[25] <span class="element-citation">Jutten C, Herault J. Blind separation of sources i. an adaptive algorithm based on neuromimetic architecture. <span><span class="ref-journal">Signal Processing. </span>1991;<span class="ref-vol">24</span>(1):1–10.</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">[26] <span class="element-citation">Lappalainen H, Miskin JW.  Ensemble learning. In: Girolami M, editor. <span class="ref-journal">Advances in Independent Component Analysis.</span> Springer-Verlag; New York: 2000. pp. 76–92.</span></div><div class="ref-cit-blk half_rhythm" id="R27">[27] <span class="element-citation">Laughlin S. A simple coding procedure enhances a neuron's information capacity. <span><span class="ref-journal">Z. Naturforsch. </span>1981;<span class="ref-vol">36</span>:910–912.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7303823" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">[28] <span class="element-citation">Lee DD, Seung S. Learning the parts of objects by nonnegative matrix factorization. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">401</span>:788–791.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10548103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">[29] <span class="element-citation">Lee T-W.  <span class="ref-journal">Independent Component Analysis: Theory and Applications.</span> Kluwer; Boston, MA: 1998. </span></div><div class="ref-cit-blk half_rhythm" id="R30">[30] <span class="element-citation">Lee T-W, Girolami M, Sejnowski TJ. Independent component analysis using an extended infomax algorithm for mixed sub-Gaussian and super-Gaussian sources. <span><span class="ref-journal">Neural Comput. </span>1999;<span class="ref-vol">11</span>(2):417–41.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9950738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">[31] <span class="element-citation">Lee T-W, Koehler BU, Orglmeister R. Blind source separation of nonlinear mixing models. Proc. IEEE Int. Workshop Neural Networks Signal Processing; Sept. 1997.pp. 406–415.</span></div><div class="ref-cit-blk half_rhythm" id="R32">[32] <span class="element-citation">Lewicki M, Olshausen B. Probabilistic framework for the adaptation and comparison of image codes. <span><span class="ref-journal">J. Opt. Soc. Amer. A. </span>1999;<span class="ref-vol">16</span>(7):1587–601.</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">[33] <span class="element-citation">Lewicki M, Sejnowski TJ. Learning overcomplete representations. <span><span class="ref-journal">Neural Comput. </span>2000;<span class="ref-vol">12</span>(2):337–65.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10636946" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">[34] <span class="element-citation">Lin J, Grier DG, Cowan J.  Source separation and density estimation by faithful equivariant som. In: Mozer M, Jordan M, Petsche T, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 9. MIT Press; Cambridge, MA: 1997. pp. 536–541.</span></div><div class="ref-cit-blk half_rhythm" id="R35">[35] <span class="element-citation">Liu C, Wechsler H. Comparative assessment of independent component analysis (ICA) for face recognition. Int. Conf. Audio Video Based Biometric Person Authentication.1999. </span></div><div class="ref-cit-blk half_rhythm" id="R36">[36] <span class="element-citation">MacKay DJC.  <span class="ref-journal">Maximum Likelihood and Covariant Algorithms for Independent Component Analysis.</span> 1996. </span></div><div class="ref-cit-blk half_rhythm" id="R37">[37] <span class="element-citation">Makeig S, Bell AJ, Jung T-P, Sejnowski TJ.  Independent component analysis of electroencephalographic data. In: Touretzky D, Mozer M, Hasselmo M, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 8. MIT Press; Cambridge, MA: 1996. pp. 145–151.</span></div><div class="ref-cit-blk half_rhythm" id="R38">[38] <span class="element-citation">Marks TK, Movellan JR. Diffusion networks, products of experts, and factor analysis; Proc. 3rd Int. Conf. Independent Component Anal. Signal Separation; 2001. </span></div><div class="ref-cit-blk half_rhythm" id="R39">[39] <span class="element-citation">McKeown MJ, Makeig S, Brown GG, Jung T-P, Kindermann SS, Bell AJ, Sejnowski TJ. Analysis of fMRI by decomposition into independent spatial components. <span><span class="ref-journal">Human Brain Mapping. </span>1998;<span class="ref-vol">6</span>(3):160–88.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9673671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">[40] <span class="element-citation">Miskin JW, MacKay DJC.  <span class="ref-journal">Ensemble Learning for Blind Source Separation ICA: Principles and Practice.</span> Cambridge Univ. Press; Cambridge, U.K.: 2001. </span></div><div class="ref-cit-blk half_rhythm" id="R41">[41] <span class="element-citation">Moghaddam B. Principal manifolds and Bayesian subspaces for visual recognition. Int. Conf. Comput. Vision.1999. </span></div><div class="ref-cit-blk half_rhythm" id="R42">[42] <span class="element-citation">Nadal J-P, Parga N. Non-linear neurons in the low noise limit: A factorial code maximizes information transfer. <span><span class="ref-journal">Network. </span>1994;<span class="ref-vol">5</span>:565–581.</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">[43] <span class="element-citation">Olshausen BA, Field DJ. Emergence of simple-cell receptive field properties by learning a sparse code for natural images. <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">381</span>:607–609.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8637596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">[44] <span class="element-citation">Olshausen BA, Field DJ. Natural image statistics and efficient coding. <span><span class="ref-journal">Network: Comput. Neural Syst. </span>1996;<span class="ref-vol">7</span>(2):333–340.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16754394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">[45] <span class="element-citation">Oppenheim AV, Lim JS. The importance of phase in signals. <span><span class="ref-journal">Proc. IEEE. </span>1981;<span class="ref-vol">69</span>:529–541.</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">[46] <span class="element-citation">O'Toole A, Deffenbacher K, Valentin D, Abdi H. Structural aspects of face recognition and the other race effect. <span><span class="ref-journal">Memory Cognition. </span>1994;<span class="ref-vol">22</span>(2):208–224.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8035697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">[47] <span class="element-citation">Palm G. On associative memory. <span><span class="ref-journal">Biol. Cybern. </span>1980;<span class="ref-vol">36</span>:19–31.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7353062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">[48] <span class="element-citation">Pearlmutter BA, Parra LC.  A context-sensitive generalization of ICA. In: Mozer Jordan, Petsche, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 9. MIT Press; Cambridge, MA: 1996. </span></div><div class="ref-cit-blk half_rhythm" id="R49">[49] <span class="element-citation">Penev PS.  Redundancy and dimensionality reduction in sparse-distributed representations of natural objects in terms of their local features. In: Leen TK, Dietterich TG, Tresp V, editors. <span class="ref-journal">Advances in Neural Information Processing Systems 13.</span> MIT Press; Cambridge, MA: 2001. </span></div><div class="ref-cit-blk half_rhythm" id="R50">[50] <span class="element-citation">Penev PS, Atick JJ. Local feature analysis: A general statistical theory for object representation. <span><span class="ref-journal">Network: Comput. Neural Syst. </span>1996;<span class="ref-vol">7</span>(3):477–500.</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">[51] <span class="element-citation">Pentland A, Moghaddam B, Starner T. View-based and modular eigenspaces for face recognition. Proc. IEEE Conf. Comput. Vision Pattern Recognition.1994. pp. 84–91.</span></div><div class="ref-cit-blk half_rhythm" id="R52">[52] <span class="element-citation">Phillips PJ, Wechsler H, Juang J, Rauss PJ. The feret database and evaluation procedure for face-recognition algorithms. <span><span class="ref-journal">Image Vision Comput. J. </span>1998;<span class="ref-vol">16</span>(5):295–306.</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">[53] <span class="element-citation">Piotrowski LN, Campbell FW. A demonstration of the visual importance and flexibility of spatial-frequency, amplitude, and phase. <span><span class="ref-journal">Perception. </span>1982;<span class="ref-vol">11</span>:337–346.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7167342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">[54] <span class="element-citation">Simoncelli EP. Statistical models for images: Compression, restoration and synthesis. 31st Asilomar Conference on Signals, Systems and Computers; Pacific Grove, CA. Nov. 2–5, 1997.</span></div><div class="ref-cit-blk half_rhythm" id="R55">[55] <span class="element-citation">Stone JV, Porrill J.  <span class="ref-journal">Undercomplete Independent Component Analysis for Signal Separation and Dimension Reduction, Tech. Rep.</span> Dept. Psych., Univ. Sheffield; Sheffield, U.K.: 1998. </span></div><div class="ref-cit-blk half_rhythm" id="R56">[56] <span class="element-citation">Teh YW, Hinton GE.  Rate-coded restricted boltzmann machines for face recognition. In: Leen T, Dietterich T, Tresp V, editors. <span class="ref-journal">Advances in Neural Information Processing Systems 13.</span> MIT Press; Cambridge, MA: 2001. </span></div><div class="ref-cit-blk half_rhythm" id="R57">[57] <span class="element-citation">Turk M, Pentland A. Eigenfaces for recognition. <span><span class="ref-journal">J. Cognitive Neurosci. </span>1991;<span class="ref-vol">3</span>(1):71–86.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/23964806" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">[58] <span class="element-citation">Wachtler T, Lee T-W, Sejnowski TJ. The chromatic structure of natural scenes. <span><span class="ref-journal">J. Opt. Soc. Amer. A. </span>2001;<span class="ref-vol">18</span>(1):65–77.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11152005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">[59] <span class="element-citation">Yang HH, Amari S-I, Cichocki A. Nformation-theoretic approach to blind separation of sources in nonlinear mixture. <span><span class="ref-journal">Signal Processing. </span>1998;<span class="ref-vol">64</span>(3):291–3000.</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">[60] <span class="element-citation">Yang M.  Face recognition using kernel methods. In: Diederich T, Becker S, Ghahramani Z, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 14 2002. </span></div><div class="ref-cit-blk half_rhythm" id="R61">[61] <span class="element-citation">Yuen PC, Lai JH. Independent component analysis of face images. IEEE Workshop Biologically Motivated Computer Vision; Seoul, Korea. 2000. </span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#idm140094232838208title">Abstract</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S1title">I. Introduction</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S2title">II. ICA</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S7title">III. Image Data</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S8title">IV. Architecture I: Statistically Independent Basis Images</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S11title">V. Architecture II: A Factorial Face Code</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S13title">VI. Examination Of The ICA Representations</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S16title">VII. Combined ICA Recognition System</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S17title">VIII. Discussion</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#S18title">Acknowledgment</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#d35e3003title">Biographies</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#idm140094242859488title">Footnotes</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#idm140094221320112title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/epub/">ePub (beta)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/pdf/nihms-199383.pdf">PDF (1.3M)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" data-citationid="PMC2898524" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Face Recognition by Independent Component Analysis_files/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F&amp;text=Face%20Recognition%20by%20Independent%20Component%20Analysis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Face Recognition by Independent Component Analysis_files/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Face Recognition by Independent Component Analysis_files/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18244540" data-aiid="2898524" data-aid="2898524" data-iid="188362" data-domainid="544" data-domain="hhmipa" data-accid="PMC2898524" data-md5="8ccab81e6cbbf4272937417fe3fe0453">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="2898524"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:&#39;#favUL&#39;,starExists:true,createCollectionUrl:&#39;#&#39;,manageCollectionsUrl:&#39;/myncbi/collections/&#39;"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="https://www.ncbi.nlm.nih.gov/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/17702297" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Constructing PCA baseline algorithms to reevaluate ICA-based face-recognition performance.</a><span class="source">[IEEE Trans Syst Man Cybern B C...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Yang J, Zhang D, Yang JY. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Syst Man Cybern B Cybern. 2007 Aug; 37(4):1015-21. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/16355663" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Effective representation using ICA for face recognition robust to local distortion and partial occlusion.</a><span class="source">[IEEE Trans Pattern Anal Mach I...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Kim J, Choi J, Yi J, Turk M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Pattern Anal Mach Intell. 2005 Dec; 27(12):1977-81. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/15376857" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Enhanced independent component analysis and its application to content based face image retrieval.</a><span class="source">[IEEE Trans Syst Man Cybern B C...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Liu C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Syst Man Cybern B Cybern. 2004 Apr; 34(2):1117-27. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/28781592" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Patch-Based Principal Component Analysis for Face Recognition.</a><span class="source">[Comput Intell Neurosci. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Jiang TX, Huang TZ, Zhao XL, Ma TH. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Comput Intell Neurosci. 2017; 2017:5317850. Epub 2017 Jul 11.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/10085426" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Feature extraction through LOCOCODE.</a><span class="source">[Neural Comput. 1999]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Hochreiter S, Schmidhuber J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Neural Comput. 1999 Apr 1; 11(3):679-714. </em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=18244540&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=18244540&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856132/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Facial Emotion Recognition: A Survey and Real-World User Experiences in Mixed Reality</a><span class="source">[Sensors (Basel, Switzerland). ...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Mehta D, Siddiqui MF, Javaid AY. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Sensors (Basel, Switzerland). 2018 Feb 1; 18(2): 416</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5549761/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A mixture of sparse coding models explaining properties of face neurons related to holistic and parts-based processing</a><span class="source">[PLoS Computational Biology. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Hosoya H, Hyvärinen A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS Computational Biology. 2017 Jul 25; 13(7): e1005667</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5385541/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Maxdenominator Reweighted Sparse Representation for Tumor Classification</a><span class="source">[Scientific Reports. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Li W, Liao B, Zhu W, Chen M, Peng L, Wei X, Gu C, Li K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Scientific Reports. 2017 Apr 10; 7: 46030</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5054104/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene Expression Data Classification Using Consensus Independent Component Analysis</a><span class="source"></span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Zheng CH, Huang DS, Kong XZ, Zhao XM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genomics, Proteomics &amp; Bioinformatics. 2008 Oct 28; 6(2): 74-82</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5023014/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Two-dimensional Whitening Reconstruction for Enhancing Robustness of Principal Component Analysis</a><span class="source">[IEEE transactions on pattern a...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Shi X, Guo Z, Nie F, Yang L, You J, Tao D. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE transactions on pattern analysis and machine intelligence. 2015 Nov 18; 38(10): 2130-2136</em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/citedby/" ref="reftype=relart&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-17" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=2898524" ref="reftype=MedGen&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/18244540/" ref="reftype=PubMed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=2898524" ref="reftype=Taxonomy&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-21" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-21" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState(&#39;ClearHT&#39;)">Clear</a><a href="javascript:historyDisplayState(&#39;HTOff&#39;)" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState(&#39;HTOn&#39;)" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5acb283a714d4527c88c2ab8">Face Recognition by Independent Component Analysis</a><div class="ralinkpop offscreen_noflow">Face Recognition by Independent Component Analysis<div class="brieflinkpopdesc">Howard Hughes Medical Institute Author Manuscripts. 2002; 13(6)1450</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5ac8f1e8cc5a91881429c4b8">Eigenfaces for recognition.</a><div class="ralinkpop offscreen_noflow">Eigenfaces for recognition.<div class="brieflinkpopdesc">J Cogn Neurosci. 1991 Winter;3(1):71-86. doi: 10.1162/jocn.1991.3.1.71.</div>
</div>
<div class="tertiary">PubMed</div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState(&#39;HTOn&#39;)">Turn recording back on</a></p>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 370px; top: 1290px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-22"><ul>
<li class="two_line" reference_id="R57">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23964806/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Eigenfaces for recognition.</a><span class="one_line_source">[J Cogn Neurosci.  1991]</span><div class="alt-note">
<div class="authors">Turk M, Pentland A</div>
<div class="citation">J Cogn Neurosci. 1991 Winter; 3(1):71-86.</div>
</div>
</li>
<li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
<li class="two_line" reference_id="R39">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9673671/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Analysis of fMRI data by blind separation into independent spatial components.</a><span class="one_line_source">[Hum Brain Mapp.  1998]</span><div class="alt-note">
<div class="authors">McKeown MJ, Makeig S, Brown GG, Jung TP, Kindermann SS, Bell AJ, Sejnowski TJ</div>
<div class="citation">Hum Brain Mapp. 1998; 6(3):160-88.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P7" name="crb--P7" rid="P7" style="position: absolute; visibility: visible; width: 370px; top: 1608px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul><li class="two_line" reference_id="R57">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23964806/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Eigenfaces for recognition.</a><span class="one_line_source">[J Cogn Neurosci.  1991]</span><div class="alt-note">
<div class="authors">Turk M, Pentland A</div>
<div class="citation">J Cogn Neurosci. 1991 Winter; 3(1):71-86.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P8" name="crb--P8" rid="P8" style="position: absolute; visibility: visible; width: 370px; top: 1743px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul>
<li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li>
<li class="two_line" reference_id="R27">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7303823/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A simple coding procedure enhances a neuron's information capacity.</a><span class="one_line_source">[Z Naturforsch C.  1981]</span><div class="alt-note">
<div class="authors">Laughlin S</div>
<div class="citation">Z Naturforsch C. 1981 Sep-Oct; 36(9-10):910-2.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P9" name="crb--P9" rid="P9" style="position: absolute; visibility: visible; width: 370px; top: 2464px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
<li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P10" name="crb--P10" rid="P10" style="position: absolute; visibility: visible; width: 370px; top: 2622px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P12" name="crb--P12" rid="P12" style="position: absolute; visibility: visible; width: 370px; top: 3388px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
<li class="two_line" reference_id="R53">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7167342/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A demonstration of the visual importance and flexibility of spatial-frequency amplitude and phase.</a><span class="one_line_source">[Perception.  1982]</span><div class="alt-note">
<div class="authors">Piotrowski LN, Campbell FW</div>
<div class="citation">Perception. 1982; 11(3):337-46.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P13" name="crb--P13" rid="P13" style="position: absolute; visibility: visible; width: 370px; top: 4260px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul><li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P18" name="crb--P18" rid="P18" style="position: absolute; visibility: visible; width: 370px; top: 5842px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul><li class="two_line" reference_id="R30">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9950738/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Independent component analysis using an extended infomax algorithm for mixed subgaussian and supergaussian sources.</a><span class="one_line_source">[Neural Comput.  1999]</span><div class="alt-note">
<div class="authors">Lee TW, Girolami M, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1999 Feb 15; 11(2):417-41.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="expanded" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li>
<li class="expanded" reference_id="R39">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9673671/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Analysis of fMRI data by blind separation into independent spatial components.</a><span class="one_line_source">[Hum Brain Mapp.  1998]</span><div class="alt-note">
<div class="authors">McKeown MJ, Makeig S, Brown GG, Jung TP, Kindermann SS, Bell AJ, Sejnowski TJ</div>
<div class="citation">Hum Brain Mapp. 1998; 6(3):160-88.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P22" name="crb--P22" rid="P22" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="expanded" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P28" name="crb--P28" rid="P28" style="position: absolute; visibility: visible; width: 370px; top: 9172px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul><li class="two_line" reference_id="R39">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9673671/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Analysis of fMRI data by blind separation into independent spatial components.</a><span class="one_line_source">[Hum Brain Mapp.  1998]</span><div class="alt-note">
<div class="authors">McKeown MJ, Makeig S, Brown GG, Jung TP, Kindermann SS, Bell AJ, Sejnowski TJ</div>
<div class="citation">Hum Brain Mapp. 1998; 6(3):160-88.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P36" name="crb--P36" rid="P36" style="position: absolute; visibility: visible; width: 370px; top: 12640px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="R23" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/1504027/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Normalization of cell responses in cat striate cortex.</a><span class="one_line_source">[Vis Neurosci.  1992]</span><div class="alt-note">
<div class="authors">Heeger DJ</div>
<div class="citation">Vis Neurosci. 1992 Aug; 9(2):181-97.</div>
</div>
</li>
<li class="two_line" reference_id="R24" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/2006233/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Lesioning an attractor network: investigations of acquired dyslexia.</a><span class="one_line_source">[Psychol Rev.  1991]</span><div class="alt-note">
<div class="authors">Hinton GE, Shallice T</div>
<div class="citation">Psychol Rev. 1991 Jan; 98(1):74-95.</div>
</div>
</li>
<li class="two_line" reference_id="R46" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8035697/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Structural aspects of face recognition and the other-race effect.</a><span class="one_line_source">[Mem Cognit.  1994]</span><div class="alt-note">
<div class="authors">O'Toole AJ, Deffenbacher KA, Valentin D, Abdi H</div>
<div class="citation">Mem Cognit. 1994 Mar; 22(2):208-24.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/" class="seemore">See links ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P43" name="crb--P43" rid="P43" style="position: absolute; visibility: visible; width: 370px; top: 15377px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul><li class="two_line" reference_id="R2">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22149669/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Could information theory provide an ecological theory of sensory processing?</a><span class="one_line_source">[Network.  2011]</span><div class="alt-note">
<div class="authors">Atick JJ</div>
<div class="citation">Network. 2011; 22(1-4):4-44.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P53" name="crb--P53" rid="P53" style="position: absolute; visibility: visible; width: 370px; top: 21504px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="R47">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7353062/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">On associative memory.</a><span class="one_line_source">[Biol Cybern.  1980]</span><div class="alt-note">
<div class="authors">Palm G</div>
<div class="citation">Biol Cybern. 1980; 36(1):19-31.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P59" name="crb--P59" rid="P59" style="position: absolute; visibility: visible; width: 370px; top: 24676px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="R57">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23964806/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Eigenfaces for recognition.</a><span class="one_line_source">[J Cogn Neurosci.  1991]</span><div class="alt-note">
<div class="authors">Turk M, Pentland A</div>
<div class="citation">J Cogn Neurosci. 1991 Winter; 3(1):71-86.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P63" name="crb--P63" rid="P63" style="position: absolute; visibility: visible; width: 370px; top: 25267px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="R2">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22149669/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Could information theory provide an ecological theory of sensory processing?</a><span class="one_line_source">[Network.  2011]</span><div class="alt-note">
<div class="authors">Atick JJ</div>
<div class="citation">Network. 2011; 22(1-4):4-44.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P65" name="crb--P65" rid="P65" style="position: absolute; visibility: visible; width: 370px; top: 25772px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An information-maximization approach to blind separation and blind deconvolution.</a><span class="one_line_source">[Neural Comput.  1995]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div>
</div>
</li>
<li class="two_line" reference_id="R30">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9950738/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Independent component analysis using an extended infomax algorithm for mixed subgaussian and supergaussian sources.</a><span class="one_line_source">[Neural Comput.  1999]</span><div class="alt-note">
<div class="authors">Lee TW, Girolami M, Sejnowski TJ</div>
<div class="citation">Neural Comput. 1999 Feb 15; 11(2):417-41.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P67" name="crb--P67" rid="P67" style="position: absolute; visibility: visible; width: 370px; top: 26172px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul><li class="two_line" reference_id="R16">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21188284/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Classifying Facial Actions.</a><span class="one_line_source">[IEEE Trans Pattern Anal Mach Intell.  1999]</span><div class="alt-note">
<div class="authors">Donato G, Bartlett MS, Hager JC, Ekman P, Sejnowski TJ</div>
<div class="citation">IEEE Trans Pattern Anal Mach Intell. 1999 Oct; 21(10):974.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P68" name="crb--P68" rid="P68" style="position: absolute; visibility: visible; width: 370px; top: 26330px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul><li class="two_line" reference_id="R28">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/10548103/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Learning the parts of objects by non-negative matrix factorization.</a><span class="one_line_source">[Nature.  1999]</span><div class="alt-note">
<div class="authors">Lee DD, Seung HS</div>
<div class="citation">Nature. 1999 Oct 21; 401(6755):788-91.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P70" name="crb--P70" rid="P70" style="position: absolute; visibility: visible; width: 370px; top: 26753px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P71" name="crb--P71" rid="P71" style="position: absolute; visibility: visible; width: 370px; top: 26890px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22149669/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Could information theory provide an ecological theory of sensory processing?</a><span class="one_line_source">[Network.  2011]</span><div class="alt-note">
<div class="authors">Atick JJ</div>
<div class="citation">Network. 2011; 22(1-4):4-44.</div>
</div>
</li>
<li class="two_line" reference_id="R43">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8637596/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Emergence of simple-cell receptive field properties by learning a sparse code for natural images.</a><span class="one_line_source">[Nature.  1996]</span><div class="alt-note">
<div class="authors">Olshausen BA, Field DJ</div>
<div class="citation">Nature. 1996 Jun 13; 381(6583):607-9.</div>
</div>
</li>
<li class="two_line" reference_id="R44">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16754394/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Natural image statistics and efficient coding.</a><span class="one_line_source">[Network.  1996]</span><div class="alt-note">
<div class="authors">Olshausen BA, Field DJ</div>
<div class="citation">Network. 1996 May; 7(2):333-9.</div>
</div>
</li>
<li class="two_line" reference_id="R18">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/3430225/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Relations between the statistics of natural images and the response properties of cortical cells.</a><span class="one_line_source">[J Opt Soc Am A.  1987]</span><div class="alt-note">
<div class="authors">Field DJ</div>
<div class="citation">J Opt Soc Am A. 1987 Dec; 4(12):2379-94.</div>
</div>
</li>
<li class="two_line" reference_id="R16">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21188284/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Classifying Facial Actions.</a><span class="one_line_source">[IEEE Trans Pattern Anal Mach Intell.  1999]</span><div class="alt-note">
<div class="authors">Donato G, Bartlett MS, Hager JC, Ekman P, Sejnowski TJ</div>
<div class="citation">IEEE Trans Pattern Anal Mach Intell. 1999 Oct; 21(10):974.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P72" name="crb--P72" rid="P72" style="position: absolute; visibility: visible; width: 370px; top: 27195px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul>
<li class="two_line" reference_id="R33">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/10636946/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Learning overcomplete representations.</a><span class="one_line_source">[Neural Comput.  2000]</span><div class="alt-note">
<div class="authors">Lewicki MS, Sejnowski TJ</div>
<div class="citation">Neural Comput. 2000 Feb; 12(2):337-65.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The "independent components" of natural scenes are edge filters.</a><span class="one_line_source">[Vision Res.  1997]</span><div class="alt-note">
<div class="authors">Bell AJ, Sejnowski TJ</div>
<div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div>
</div>
</li>
<li class="two_line" reference_id="R43">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8637596/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Emergence of simple-cell receptive field properties by learning a sparse code for natural images.</a><span class="one_line_source">[Nature.  1996]</span><div class="alt-note">
<div class="authors">Olshausen BA, Field DJ</div>
<div class="citation">Nature. 1996 Jun 13; 381(6583):607-9.</div>
</div>
</li>
<li class="two_line" reference_id="R58" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11152005/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Chromatic structure of natural scenes.</a><span class="one_line_source">[J Opt Soc Am A Opt Image Sci Vis.  2001]</span><div class="alt-note">
<div class="authors">Wachtler T, Lee TW, Sejnowski TJ</div>
<div class="citation">J Opt Soc Am A Opt Image Sci Vis. 2001 Jan; 18(1):65-77.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/" class="seemore">See more ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p17" name="crb--__p17" rid="__p17" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul>
<li class="expanded" reference_id="R43">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8637596/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Emergence of simple-cell receptive field properties by learning a sparse code for natural images.</a><span class="one_line_source">[Nature.  1996]</span><div class="alt-note">
<div class="authors">Olshausen BA, Field DJ</div>
<div class="citation">Nature. 1996 Jun 13; 381(6583):607-9.</div>
</div>
</li>
<li class="expanded" reference_id="R44">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16754394/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Natural image statistics and efficient coding.</a><span class="one_line_source">[Network.  1996]</span><div class="alt-note">
<div class="authors">Olshausen BA, Field DJ</div>
<div class="citation">Network. 1996 May; 7(2):333-9.</div>
</div>
</li>
</ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./Face Recognition by Independent Component Analysis_files/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-R5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[5] <span class="element-citation">Barlow HB. Unsupervised learning. <span><span class="ref-journal">Neural Comput. </span>1989;<span class="ref-vol">1</span>:295–311.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R5">Ref list</a>]</div><div id="body-link-popper-R3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[3] <span class="element-citation">Atick JJ, Redlich AN. What does the retina know about natural scenes? <span><span class="ref-journal">Neural Comput. </span>1992;<span class="ref-vol">4</span>:196–210.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R3">Ref list</a>]</div><div id="body-link-popper-R57" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Eigenfaces for recognition.<div class="authors">Turk M, Pentland A</div><div class="citation">J Cogn Neurosci. 1991 Winter; 3(1):71-86.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/23964806/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R57">Ref list</a>]</p></div><div id="body-link-popper-R15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[15] <span class="element-citation">Cottrell G, Metcalfe J.  Face, gender and emotion recognition using holons. In: Touretzky D, editor. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 3. Morgan Kaufmann; San Mateo, CA: 1991. pp. 564–571.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R15">Ref list</a>]</div><div id="body-link-popper-R50" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[50] <span class="element-citation">Penev PS, Atick JJ. Local feature analysis: A general statistical theory for object representation. <span><span class="ref-journal">Network: Comput. Neural Syst. </span>1996;<span class="ref-vol">7</span>(3):477–500.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R50">Ref list</a>]</div><div id="body-link-popper-R14" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[14] <span class="element-citation">Comon P. Independent component analysis—A new concept? <span><span class="ref-journal">Signal Processing. </span>1994;<span class="ref-vol">36</span>:287–314.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R14">Ref list</a>]</div><div id="body-link-popper-R20" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[20] <span class="element-citation">Girolami M.  <span class="ref-journal">Advances in Independent Component Analysis.</span> Springer-Verlag; Berlin, Germany: 2000. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R20">Ref list</a>]</div><div id="body-link-popper-R29" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[29] <span class="element-citation">Lee T-W.  <span class="ref-journal">Independent Component Analysis: Theory and Applications.</span> Kluwer; Boston, MA: 1998. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R29">Ref list</a>]</div><div id="body-link-popper-R11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">An information-maximization approach to blind separation and blind deconvolution.<div class="authors">Bell AJ, Sejnowski TJ</div><div class="citation">Neural Comput. 1995 Nov; 7(6):1129-59.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7584893/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R11">Ref list</a>]</p></div><div id="body-link-popper-R12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The "independent components" of natural scenes are edge filters.<div class="authors">Bell AJ, Sejnowski TJ</div><div class="citation">Vision Res. 1997 Dec; 37(23):3327-38.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9425547/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R12">Ref list</a>]</p></div><div id="body-link-popper-R37" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[37] <span class="element-citation">Makeig S, Bell AJ, Jung T-P, Sejnowski TJ.  Independent component analysis of electroencephalographic data. In: Touretzky D, Mozer M, Hasselmo M, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 8. MIT Press; Cambridge, MA: 1996. pp. 145–151.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R37">Ref list</a>]</div><div id="body-link-popper-R39" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Analysis of fMRI data by blind separation into independent spatial components.<div class="authors">McKeown MJ, Makeig S, Brown GG, Jung TP, Kindermann SS, Bell AJ, Sejnowski TJ</div><div class="citation">Hum Brain Mapp. 1998; 6(3):160-88.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9673671/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R39">Ref list</a>]</p></div><div id="body-link-popper-R52" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[52] <span class="element-citation">Phillips PJ, Wechsler H, Juang J, Rauss PJ. The feret database and evaluation procedure for face-recognition algorithms. <span><span class="ref-journal">Image Vision Comput. J. </span>1998;<span class="ref-vol">16</span>(5):295–306.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R52">Ref list</a>]</div><div id="body-link-popper-R51" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[51] <span class="element-citation">Pentland A, Moghaddam B, Starner T. View-based and modular eigenspaces for face recognition. Proc. IEEE Conf. Comput. Vision Pattern Recognition.1994. pp. 84–91.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R51">Ref list</a>]</div><div id="body-link-popper-R13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[13] <span class="element-citation">Cichocki A, Unbehauen R, Rummert E. Robust learning algorithm for blind separation of signals. <span><span class="ref-journal">Electron. Lett. </span>1994;<span class="ref-vol">30</span>(7):1386–1387.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R13">Ref list</a>]</div><div id="body-link-popper-R25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[25] <span class="element-citation">Jutten C, Herault J. Blind separation of sources i. an adaptive algorithm based on neuromimetic architecture. <span><span class="ref-journal">Signal Processing. </span>1991;<span class="ref-vol">24</span>(1):1–10.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R25">Ref list</a>]</div><div id="body-link-popper-R27" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A simple coding procedure enhances a neuron's information capacity.<div class="authors">Laughlin S</div><div class="citation">Z Naturforsch C. 1981 Sep-Oct; 36(9-10):910-2.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7303823/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R27">Ref list</a>]</p></div><div id="body-link-popper-R1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[1] <span class="element-citation">Amari S, Cichocki A, Yang HH.  <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 8. MIT Press; Cambridge, MA: 1996. A new learning algorithm for blind signal separation.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R1">Ref list</a>]</div><div id="body-link-popper-R42" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[42] <span class="element-citation">Nadal J-P, Parga N. Non-linear neurons in the low noise limit: A factorial code maximizes information transfer. <span><span class="ref-journal">Network. </span>1994;<span class="ref-vol">5</span>:565–581.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R42">Ref list</a>]</div><div id="body-link-popper-R36" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[36] <span class="element-citation">MacKay DJC.  <span class="ref-journal">Maximum Likelihood and Covariant Algorithms for Independent Component Analysis.</span> 1996. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R36">Ref list</a>]</div><div id="body-link-popper-R48" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[48] <span class="element-citation">Pearlmutter BA, Parra LC.  A context-sensitive generalization of ICA. In: Mozer Jordan, Petsche, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 9. MIT Press; Cambridge, MA: 1996. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R48">Ref list</a>]</div><div id="body-link-popper-R19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[19] <span class="element-citation">Field DJ. What is the goal of sensory coding? <span><span class="ref-journal">Neural Comput. </span>1994;<span class="ref-vol">6</span>:559–601.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R19">Ref list</a>]</div><div id="body-link-popper-R45" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[45] <span class="element-citation">Oppenheim AV, Lim JS. The importance of phase in signals. <span><span class="ref-journal">Proc. IEEE. </span>1981;<span class="ref-vol">69</span>:529–541.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R45">Ref list</a>]</div><div id="body-link-popper-R53" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A demonstration of the visual importance and flexibility of spatial-frequency amplitude and phase.<div class="authors">Piotrowski LN, Campbell FW</div><div class="citation">Perception. 1982; 11(3):337-46.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7167342/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R53">Ref list</a>]</p></div><div id="body-link-popper-R30" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Independent component analysis using an extended infomax algorithm for mixed subgaussian and supergaussian sources.<div class="authors">Lee TW, Girolami M, Sejnowski TJ</div><div class="citation">Neural Comput. 1999 Feb 15; 11(2):417-41.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9950738/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R30">Ref list</a>]</p></div><div id="body-link-popper-R23" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Normalization of cell responses in cat striate cortex.<div class="authors">Heeger DJ</div><div class="citation">Vis Neurosci. 1992 Aug; 9(2):181-97.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/1504027/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R23">Ref list</a>]</p></div><div id="body-link-popper-R24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Lesioning an attractor network: investigations of acquired dyslexia.<div class="authors">Hinton GE, Shallice T</div><div class="citation">Psychol Rev. 1991 Jan; 98(1):74-95.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/2006233/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R24">Ref list</a>]</p></div><div id="body-link-popper-R46" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Structural aspects of face recognition and the other-race effect.<div class="authors">O'Toole AJ, Deffenbacher KA, Valentin D, Abdi H</div><div class="citation">Mem Cognit. 1994 Mar; 22(2):208-24.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8035697/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R46">Ref list</a>]</p></div><div id="body-link-popper-R2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Could information theory provide an ecological theory of sensory processing?<div class="authors">Atick JJ</div><div class="citation">Network. 2011; 22(1-4):4-44.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/22149669/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R2">Ref list</a>]</p></div><div id="body-link-popper-R43" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[43] <span class="element-citation">Olshausen BA, Field DJ. Emergence of simple-cell receptive field properties by learning a sparse code for natural images. <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">381</span>:607–609.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8637596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R43">Ref list</a>]</div><div id="body-link-popper-R44" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[44] <span class="element-citation">Olshausen BA, Field DJ. Natural image statistics and efficient coding. <span><span class="ref-journal">Network: Comput. Neural Syst. </span>1996;<span class="ref-vol">7</span>(2):333–340.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16754394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R44">Ref list</a>]</div><div id="body-link-popper-R10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[10] <span class="element-citation">Baum EB, Moody J, Wilczek F. Internal representaions for associative memory. <span><span class="ref-journal">Biol. Cybern. </span>1988;<span class="ref-vol">59</span>:217–228.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R10">Ref list</a>]</div><div id="body-link-popper-R47" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">On associative memory.<div class="authors">Palm G</div><div class="citation">Biol Cybern. 1980; 36(1):19-31.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7353062/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R47">Ref list</a>]</p></div><div id="body-link-popper-R41" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[41] <span class="element-citation">Moghaddam B. Principal manifolds and Bayesian subspaces for visual recognition. Int. Conf. Comput. Vision.1999. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R41">Ref list</a>]</div><div id="body-link-popper-R9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[9] <span class="element-citation">Bartlett MS, Lades HM, Sejnowski TJ. Independent component representations for face recognition. In: Rogowitz T, Pappas B, editors. Proc. SPIE Symp. Electon. Imaging: Science Technology—Human Vision and Electronic Imaging III; San Jose, CA. 1998. pp. 528–539.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R9">Ref list</a>]</div><div id="body-link-popper-R35" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[35] <span class="element-citation">Liu C, Wechsler H. Comparative assessment of independent component analysis (ICA) for face recognition. Int. Conf. Audio Video Based Biometric Person Authentication.1999. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R35">Ref list</a>]</div><div id="body-link-popper-R61" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[61] <span class="element-citation">Yuen PC, Lai JH. Independent component analysis of face images. IEEE Workshop Biologically Motivated Computer Vision; Seoul, Korea. 2000. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R61">Ref list</a>]</div><div id="body-link-popper-R17" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[17] <span class="element-citation">Draper BA, Baek K, Bartlett MS, Beveridge JR.  <span class="ref-journal">Comput. Vision Image Understanding (Special Issue on Face Recognition)</span> 2002. Recognizing faces with PCA and ICA. submitted for publication.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R17">Ref list</a>]</div><div id="body-link-popper-R55" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[55] <span class="element-citation">Stone JV, Porrill J.  <span class="ref-journal">Undercomplete Independent Component Analysis for Signal Separation and Dimension Reduction, Tech. Rep.</span> Dept. Psych., Univ. Sheffield; Sheffield, U.K.: 1998. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R55">Ref list</a>]</div><div id="body-link-popper-R26" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[26] <span class="element-citation">Lappalainen H, Miskin JW.  Ensemble learning. In: Girolami M, editor. <span class="ref-journal">Advances in Independent Component Analysis.</span> Springer-Verlag; New York: 2000. pp. 76–92.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R26">Ref list</a>]</div><div id="body-link-popper-R40" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[40] <span class="element-citation">Miskin JW, MacKay DJC.  <span class="ref-journal">Ensemble Learning for Blind Source Separation ICA: Principles and Practice.</span> Cambridge Univ. Press; Cambridge, U.K.: 2001. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R40">Ref list</a>]</div><div id="body-link-popper-R21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[21] <span class="element-citation">Hallinan P.  <span class="ref-journal">Ph.D. dissertation.</span> Harvard Univ.; Cambridge, MA: 1995. A Deformable Model for Face Recognition Under Arbitrary Lighting Conditions.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R21">Ref list</a>]</div><div id="body-link-popper-R31" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[31] <span class="element-citation">Lee T-W, Koehler BU, Orglmeister R. Blind source separation of nonlinear mixing models. Proc. IEEE Int. Workshop Neural Networks Signal Processing; Sept. 1997.pp. 406–415.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R31">Ref list</a>]</div><div id="body-link-popper-R59" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[59] <span class="element-citation">Yang HH, Amari S-I, Cichocki A. Nformation-theoretic approach to blind separation of sources in nonlinear mixture. <span><span class="ref-journal">Signal Processing. </span>1998;<span class="ref-vol">64</span>(3):291–3000.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R59">Ref list</a>]</div><div id="body-link-popper-R4" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[4] <span class="element-citation">Bach FR, Jordan MI. Kernel independent component analysis. <span><span class="ref-journal">J. Machine Learning Res. </span>2002;<span class="ref-vol">3</span>:1–48.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R4">Ref list</a>]</div><div id="body-link-popper-R60" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[60] <span class="element-citation">Yang M.  Face recognition using kernel methods. In: Diederich T, Becker S, Ghahramani Z, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 14 2002. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R60">Ref list</a>]</div><div id="body-link-popper-R16" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Classifying Facial Actions.<div class="authors">Donato G, Bartlett MS, Hager JC, Ekman P, Sejnowski TJ</div><div class="citation">IEEE Trans Pattern Anal Mach Intell. 1999 Oct; 21(10):974.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/21188284/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R16">Ref list</a>]</p></div><div id="body-link-popper-R49" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[49] <span class="element-citation">Penev PS.  Redundancy and dimensionality reduction in sparse-distributed representations of natural objects in terms of their local features. In: Leen TK, Dietterich TG, Tresp V, editors. <span class="ref-journal">Advances in Neural Information Processing Systems 13.</span> MIT Press; Cambridge, MA: 2001. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R49">Ref list</a>]</div><div id="body-link-popper-R28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Learning the parts of objects by non-negative matrix factorization.<div class="authors">Lee DD, Seung HS</div><div class="citation">Nature. 1999 Oct 21; 401(6755):788-91.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10548103/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R28">Ref list</a>]</p></div><div id="body-link-popper-R56" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[56] <span class="element-citation">Teh YW, Hinton GE.  Rate-coded restricted boltzmann machines for face recognition. In: Leen T, Dietterich T, Tresp V, editors. <span class="ref-journal">Advances in Neural Information Processing Systems 13.</span> MIT Press; Cambridge, MA: 2001. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R56">Ref list</a>]</div><div id="body-link-popper-R32" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[32] <span class="element-citation">Lewicki M, Olshausen B. Probabilistic framework for the adaptation and comparison of image codes. <span><span class="ref-journal">J. Opt. Soc. Amer. A. </span>1999;<span class="ref-vol">16</span>(7):1587–601.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R32">Ref list</a>]</div><div id="body-link-popper-R34" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[34] <span class="element-citation">Lin J, Grier DG, Cowan J.  Source separation and density estimation by faithful equivariant som. In: Mozer M, Jordan M, Petsche T, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 9. MIT Press; Cambridge, MA: 1997. pp. 536–541.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R34">Ref list</a>]</div><div id="body-link-popper-R38" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[38] <span class="element-citation">Marks TK, Movellan JR. Diffusion networks, products of experts, and factor analysis; Proc. 3rd Int. Conf. Independent Component Anal. Signal Separation; 2001. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R38">Ref list</a>]</div><div id="body-link-popper-R18" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Relations between the statistics of natural images and the response properties of cortical cells.<div class="authors">Field DJ</div><div class="citation">J Opt Soc Am A. 1987 Dec; 4(12):2379-94.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/3430225/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R18">Ref list</a>]</p></div><div id="body-link-popper-R54" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[54] <span class="element-citation">Simoncelli EP. Statistical models for images: Compression, restoration and synthesis. 31st Asilomar Conference on Signals, Systems and Computers; Pacific Grove, CA. Nov. 2–5, 1997.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R54">Ref list</a>]</div><div id="body-link-popper-R6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[6] <span class="element-citation">Bartlett MS.  <span class="ref-journal">Face Image Analysis by Unsupervised Learning.</span> vol. 612. Kluwer; Boston, MA: 2001. Kluwer International Series on Engineering and Computer Science.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R6">Ref list</a>]</div><div id="body-link-popper-R8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[8] <span class="element-citation">Bartlett MS, Donato GL, Movellan JR, Hager JC, Ekman P, Sejnowski TJ.  Image representations for facial expression coding. In: Solla SA, Leen TK, Muller K-R, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> vol. 12. MIT Press; Cambridge, MA: 2000. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R8">Ref list</a>]</div><div id="body-link-popper-R22" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[22] <span class="element-citation">Hancock P.  <span class="ref-journal">British Psych. Soc., Cognitive Section.</span> Univ. Essex; Essex, U.K.: 2000. Alternative representations for faces.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R22">Ref list</a>]</div><div id="body-link-popper-R33" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Learning overcomplete representations.<div class="authors">Lewicki MS, Sejnowski TJ</div><div class="citation">Neural Comput. 2000 Feb; 12(2):337-65.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10636946/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R33">Ref list</a>]</p></div><div id="body-link-popper-R58" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Chromatic structure of natural scenes.<div class="authors">Wachtler T, Lee TW, Sejnowski TJ</div><div class="citation">J Opt Soc Am A Opt Image Sci Vis. 2001 Jan; 18(1):65-77.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11152005/" ref="reftype=pubmed&amp;article-id=2898524&amp;issue-id=188362&amp;journal-id=544&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R58">Ref list</a>]</p></div><div id="body-link-popper-R7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">[7] <span class="element-citation">Bartlett MS.  <span class="ref-journal">Ph.D. dissertation.</span> Univ. California-San Diego; La Jolla: 1998. Face Image Analysis by Unsupervised Learning and Redundancy Reduction.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#R7">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2018-04-09T04:45:46-04:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=ptpmc202&amp;ncbi_phid=F4FC6255ACB279F100000000007B0079&amp;ncbi_session=396F2AEDAC8F1E71_0174SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2898524%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="https://www.ncbi.nlm.nih.gov/home/about/policies.shtml">Policies and Guidelines</a> | <a href="https://www.ncbi.nlm.nih.gov/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/InstrumentPageStarterJS.js"> </script>    
    <script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 63.5938px; left: 939.188px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898524/#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div><script async="1" src="https://www.google-analytics.com/analytics.js"></script>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript>&lt;img alt="statistics" src="/stat?jsdisabled=true&amp;amp;ncbi_db=pmc&amp;amp;ncbi_pdid=article&amp;amp;ncbi_acc=&amp;amp;ncbi_domain=hhmipa&amp;amp;ncbi_report=record&amp;amp;ncbi_type=fulltext&amp;amp;ncbi_objectid=&amp;amp;ncbi_pcid=/articles/PMC2898524/&amp;amp;ncbi_app=pmc" /&gt;</noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 396F2AEDAC8F1E71_0174SID /projects/PMC/PMCViewer@4.46 ptpmc202 v4.1.r554919 Thu, Jan 11 2018 03:16:18 -->

<script type="text/javascript" async="" id="smcx-sdk" src="./Face Recognition by Independent Component Analysis_files/tRaiETqnLgj758hTBazgd_2Fzt773GyyxFLU9m0FUrhBgVIj1hS_2F691NH_2BO1UAeHBp.js"></script><script type="text/javascript" src="./Face Recognition by Independent Component Analysis_files/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1" style="display: none;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: STIXSizeOneSym, sans-serif;"></div></div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-right" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div></body></html>